GSEA printing test 
========================================================


```{r standalone setup, echo=FALSE}
strain<-"linc-Enc1"
timepoint<-"Embryonic"
filename<-"linc-Enc1_vs_WT_Embryonic"
dir<-"/n/rinn_data1/seq/lgoff/Projects/BrainMap/data/olddiffs/linc-Enc1_vs_WT_Embryonic/"
```


```{r init, echo=FALSE}
# Set knitr opts

#path<-paste(filename,"/figure/",sep="")
opts_chunk$set(echo=FALSE, message=FALSE, warning=FALSE,fig.height=4, fig.width=4,dev=c('png', 'pdf'))
alpha<-0.05 

analysisdir<-"/n/rinn_data1/users/agroff/GITHUB/BrainMap/analysis/"
diffdir<-"/n/rinn_data1/seq/lgoff/Projects/BrainMap/data/diffs"
GTF<-"/n/rinn_data1/seq/lgoff/Projects/BrainMap/data/annotation/mm10_gencode_vM2_with_lncRNAs_and_LacZ.gtf"
genome<-"mm10"

```

```{r vars_and_setup,echo=FALSE,results='hide',message=FALSE}
library(cummeRbund)
library(limma)
library(GSA)
library(gplots)
library(marray)
library(ggplot2)
library(gtable)
library(gridExtra)
library(RColorBrewer)

reactome_gs <- GSA.read.gmt("/n/rinn_data1/users/agroff/GSEA/c2.cp.reactome.v4.0.symbols.gmt")

biocarta_gs <- GSA.read.gmt("/n/rinn_data1/users/agroff/GSEA/c2.cp.biocarta.v4.0.symbols.gmt")

kegg_gs<-GSA.read.gmt("/n/rinn_data1/annotations/gsea/msigdb_v4.0_files_to_download_locally/msigdb_v4.0_GMTs/c2.cp.kegg.v4.0.symbols.gmt")
```


```{r cummeRbund_and_data_setup, echo=FALSE}
setwd(dir)
cuff<-readCufflinks(gtfFile=GTF,genome=genome)
```


```{r gsea_helper_functions}

population<-genes(cuff)
population.diff<-diffData(population)
annotation<-annotation(genes(cuff))
gene_names<-merge(annotation,population.diff)

gene_set_index <- function(genelist, short_names){
  which(short_names %in% genelist)   
}

get_gene_set_p_vals <- function(input, gs, alternative){
  gene_set_indices <- lapply(gs$genesets, function(genelist){
    gene_set_index(genelist,input$short_name)
    })
  pvl<-lapply(gene_set_indices,geneSetTest,input$test_stat, alternative=alternative)
  pvl_mat<-as.data.frame(t(unlist(pvl)))
  colnames(pvl_mat) <- gs$geneset.names
   return(pvl_mat)
}
 
get_gene_set_q_vals <- function(pvl_mat, method="bonferroni"){
  comp_corrected <- matrix(p.adjust(pvl_mat, method=method), nrow=nrow(pvl_mat), ncol=ncol(pvl_mat))
	colnames(comp_corrected) <- colnames(pvl_mat)
	rownames(comp_corrected) <- rownames(pvl_mat)
	return(comp_corrected)
}

colMins<-function(x){
  apply(x,2,min)
}
rowMins<-function(x){
  apply(x,1,min)
}
 
InputCols<-maPalette(low="white",high="red",k=100)


ztest<-function(samp,pop){
  (mean(samp,na.rm=T)-mean(pop,na.rm=T))/sd(pop,na.rm=T)
}


get_gene_set_ztest <- function(scoring_df, gs){  
  gene_set_indices <- lapply(gs$genesets, function(genelist){
    gene_set_index(genelist, scoring_df$short_name)
    })
  
#  zscores <- apply(scoring_df$,2, function(mat_col){
#			lapply(gene_set_indices,function(gsi){
#					ztest(mat_col[gsi],mat_col)})
 #     })
  
  zscores <- lapply(gene_set_indices,function(gsi){
					ztest(scoring_df$test_stat[gsi],scoring_df$test_stat)
          })  
  
	zscore_mat<-do.call(rbind,lapply(zscores,unlist))
	rownames(zscore_mat) <- gs$geneset.names
	colnames(zscore_mat) <- "zscore"
	return(zscore_mat)
}
```

```{r gsea}
df.pop<-data.frame("short_name"=toupper(gene_names$gene_short_name),"test_stat"=gene_names$test_stat)
df.pop.unique<-unique(df.pop)
rownames(df.pop.unique)<-NULL
df.pop.unique.ordered<-df.pop.unique[order(df.pop.unique$test_stat),]
Input.df<-df.pop.unique.ordered
Input.df$short_name<-as.character(Input.df$short_name)

reactome_pvl_mat <- get_gene_set_p_vals(Input.df, reactome_gs,alternative="either")
reactome_pvl_corrected <- get_gene_set_q_vals(reactome_pvl_mat)
reactome_pvl_corrected<-rbind(reactome_pvl_corrected,reactome_pvl_corrected)

biocarta_pvl_mat <- get_gene_set_p_vals(Input.df, biocarta_gs, alternative="either")
biocarta_pvl_corrected <- get_gene_set_q_vals(biocarta_pvl_mat)
biocarta_pvl_corrected<-rbind(biocarta_pvl_corrected,biocarta_pvl_corrected)

kegg_pvl_mat <- get_gene_set_p_vals(Input.df, kegg_gs, alternative="either")
kegg_pvl_corrected <- get_gene_set_q_vals(kegg_pvl_mat)
kegg_pvl_corrected<-rbind(kegg_pvl_corrected,kegg_pvl_corrected)


reactome_zscores<-get_gene_set_ztest(Input.df,reactome_gs)
reactome_zscores<-cbind(reactome_zscores,reactome_zscores)

biocarta_zscores<-get_gene_set_ztest(Input.df,biocarta_gs)
biocarta_zscores<-cbind(biocarta_zscores,biocarta_zscores)

kegg_zscores<-get_gene_set_ztest(Input.df,kegg_gs)
kegg_zscores<-cbind(kegg_zscores,kegg_zscores)



k <- 100
myColors<-maPalette(low="blue",mid="white",high="red",k=k)
#myBreaks<-seq(-max(abs(x)),max(abs(x)),k+1)
myBreaks<-seq(-2,2,length.out=(k+1))
enrichmentBreaks<-seq(0,6,length.out=(k+1))
```


Biocarta enrichment: 

```{r print_GSEA_biocarta, fig.height=8, fig.width=8}
InputCols<-maPalette(low="white",high="red",k=100)

x<-(-log10(t(biocarta_pvl_corrected[,which(colMins(biocarta_pvl_corrected) < 0.01)])))
x_ordered<-x[order(x[,1], decreasing=TRUE),]
if(length(x_ordered)>50){x_ordered<-x_ordered[1:50,]}
noinfinities_x<-x_ordered[which(x_ordered !="Inf")]
x_max<-max(noinfinities_x)+100
x_ordered[x_ordered=="Inf"]<-x_max
x_ordered<-as.matrix(x_ordered)

if(dim(x_ordered)[1]>1){
  heatmap.2(x_ordered, trace="none",col=InputCols,breaks=enrichmentBreaks, margins=c(1,20),dendrogram="both",labCol=c(""),cexRow = 1, offsetRow=0)
}else{print("Not enough significant categories to print a heatmap!")}
```

Biocarta zscore: 

```{r gsea_zscore_biocarta, fig.height=8, fig.width=8}
x<-((biocarta_zscores[which(colMins(biocarta_pvl_corrected) < 0.01),]))
x_ordered<-x[order(x[,1], decreasing=TRUE),]
if(length(x_ordered)>50){x_ordered<-x_ordered[1:50,]}
x_ordered<-as.matrix(x_ordered)

if(dim(x_ordered)[1]>1){
heatmap.2(x_ordered, trace="none",col=myColors,breaks=myBreaks,margins=c(1,20),dendrogram="both",labCol=c(""),cexRow =1, offsetRow=0)
}else{print("Not enough significant categories to print a heatmap!")}
```


Reactome enrichment: 

```{r print_GSEA_reactome, fig.width=10, fig.height=12}
#pdf("components_GSEA_reactome.pdf",width=10,height=30)

InputCols<-maPalette(low="white",high="red",k=100)

x<-(-log10(t(reactome_pvl_corrected[,which(colMins(reactome_pvl_corrected) < 0.001)])))
x_ordered<-x[order(x[,1], decreasing=TRUE),]
if(length(x_ordered)>50){x_ordered<-x_ordered[1:50,]}
noinfinities_x<-x_ordered[which(x_ordered !="Inf")]
x_max<-max(noinfinities_x)+100
x_ordered[x_ordered=="Inf"]<-x_max

if(dim(x_ordered)[1]>1){
  heatmap.2(x_ordered, trace="none",col=InputCols,breaks=enrichmentBreaks, margins=c(1,20),dendrogram="both",labCol=c(""),cexRow =1, offsetRow=0)
}else{print("Not enough significant categories to print a heatmap!")}

#dev.off()
```


Reactome zscore: 

```{r gsea_zscore_reactome, fig.width=10, fig.height=12}

x<-reactome_zscores[which(colMins(reactome_pvl_corrected) < 0.001),]

x_ordered<-x[order(x[,1], decreasing=TRUE),]

if(length(x_ordered)>50){x_ordered<-x_ordered[1:50,]}
x_ordered<-as.matrix(x_ordered)

if(dim(x_ordered)[1]>1){
heatmap.2(x_ordered, trace="none",col=myColors,breaks=myBreaks,margins=c(1,20),dendrogram="both",labCol=c(""),cexRow = 1, offsetRow=0)
}else{print("Not enough significant categories to print a heatmap!")}
```


Kegg enrichment: 

```{r gsea_enrichment_kegg, fig.width=10, fig.height=12}
InputCols<-maPalette(low="white",high="red",k=100)

x<-(-log10(t(kegg_pvl_corrected[,which(colMins(kegg_pvl_corrected) < 0.001)])))
x_ordered<-x[order(x[,1], decreasing=TRUE),]
if(length(x_ordered)>50){x_ordered<-x_ordered[1:50,]}
noinfinities_x<-x_ordered[which(x_ordered !="Inf")]
x_max<-max(noinfinities_x)+100
x_ordered[x_ordered=="Inf"]<-x_max

if(dim(x_ordered)[1]>1){
  heatmap.2(x_ordered, trace="none",col=InputCols,breaks=enrichmentBreaks, margins=c(1,20),dendrogram="both",labCol=c(""),cexRow =1, offsetRow=0)
}else{print("Not enough significant categories to print a heatmap!")}


```

Kegg zscore: 

```{r gsea_zscore_kegg, fig.width=10, fig.height=12}

x<-kegg_zscores[which(colMins(kegg_pvl_corrected) < 0.001),]

x_ordered<-x[order(x[,1], decreasing=TRUE),]

if(length(x_ordered)>50){x_ordered<-x_ordered[1:50,]}
x_ordered<-as.matrix(x_ordered)

if(dim(x_ordered)[1]>1){
heatmap.2(x_ordered, trace="none",col=myColors,breaks=myBreaks,margins=c(1,20),dendrogram="both",labCol=c(""),cexRow = 1, offsetRow=0)
}else{print("Not enough significant categories to print a heatmap!")}

```

