<!DOCTYPE html>
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>

<title>Celrr KO vs WT (Adult)</title>





<style type="text/css">
body, td {
   font-family: sans-serif;
   background-color: white;
   font-size: 13px;
}

body {
  max-width: 800px;
  margin: auto;
  padding: 1em;
  line-height: 20px;
}

tt, code, pre {
   font-family: 'DejaVu Sans Mono', 'Droid Sans Mono', 'Lucida Console', Consolas, Monaco, monospace;
}

h1 {
   font-size:2.2em;
}

h2 {
   font-size:1.8em;
}

h3 {
   font-size:1.4em;
}

h4 {
   font-size:1.0em;
}

h5 {
   font-size:0.9em;
}

h6 {
   font-size:0.8em;
}

a:visited {
   color: rgb(50%, 0%, 50%);
}

pre, img {
  max-width: 100%;
}

pre code {
   display: block; padding: 0.5em;
}

code {
  font-size: 92%;
  border: 1px solid #ccc;
}

code[class] {
  background-color: #F8F8F8;
}

table, td, th {
  border: none;
}

blockquote {
   color:#666666;
   margin:0;
   padding-left: 1em;
   border-left: 0.5em #EEE solid;
}

hr {
   height: 0px;
   border-bottom: none;
   border-top-width: thin;
   border-top-style: dotted;
   border-top-color: #999999;
}

@media print {
   * {
      background: transparent !important;
      color: black !important;
      filter:none !important;
      -ms-filter: none !important;
   }

   body {
      font-size:12pt;
      max-width:100%;
   }

   a, a:visited {
      text-decoration: underline;
   }

   hr {
      visibility: hidden;
      page-break-before: always;
   }

   pre, blockquote {
      padding-right: 1em;
      page-break-inside: avoid;
   }

   tr, img {
      page-break-inside: avoid;
   }

   img {
      max-width: 100% !important;
   }

   @page :left {
      margin: 15mm 20mm 15mm 10mm;
   }

   @page :right {
      margin: 15mm 10mm 15mm 20mm;
   }

   p, h2, h3 {
      orphans: 3; widows: 3;
   }

   h2, h3 {
      page-break-after: avoid;
   }
}
</style>



</head>

<body>
<h1>Celrr KO vs WT (Adult)</h1>

<ul>
<li>other heatmap labels (csHeatmap)</li>
<li>cache gsea and overlap figure!! </li>
<li>tracks </li>
<li>should import litter info</li>
</ul>

<h1>Intialize</h1>

<h1>Design Overview</h1>

<p>This file shows the wt-v-ko comparison for Celrr. </p>

<p>Cuff overview:</p>

<pre><code>## CuffSet instance with:
##   2 samples
##   26754 genes
##   77524 isoforms
##   0 TSS
##   23066 CDS
##   0 promoters
##   0 splicing
##   19954 relCDS
</code></pre>

<h1>QC</h1>

<h2>Dispersion</h2>

<p>Dispersion plot for genes in cuff:
(Overdispersion can lead to innacurate quants)</p>

<pre><code>## Error: &#39;from&#39; cannot be NA, NaN or infinite
</code></pre>

<h2>Cross-replicate variability (fpkmSCVplot)</h2>

<p>Differences in CV 2 can result in lower numbers of differentially expressed genes due to a higher degree of variability between replicate fpkm estimates.</p>

<p>Genes:
<img src="figure/CV_genes.png" alt="plot of chunk CV_genes"/> </p>

<p>Isoforms: 
<img src="figure/cv_iso.png" alt="plot of chunk cv_iso"/> </p>

<h2>Volcano</h2>

<h3>Volcano matrix (replicates)</h3>

<p><img src="figure/volcano_matrix.png" alt="plot of chunk volcano_matrix"/> </p>

<h2>MvA plot</h2>

<h3>MvA plot counts</h3>

<h2>Scatterplot</h2>

<p><img src="figure/scatterplot.png" alt="plot of chunk scatterplot"/> </p>

<h3>Scatter matrix (replicates) &ndash; SKIP FOR NOW CAUSING PROBLEMS</h3>

<h2>Distributions</h2>

<h3>Boxplots</h3>

<p>Boxplot (genes)</p>

<p><img src="figure/boxplot_genes.png" alt="plot of chunk boxplot_genes"/> </p>

<p>Boxplot (genes, replicates)</p>

<p><img src="figure/boxplot_genes_replicates.png" alt="plot of chunk boxplot_genes_replicates"/> </p>

<p>Boxplot (isoforms)</p>

<p><img src="figure/boxplot_isoforms.png" alt="plot of chunk boxplot_isoforms"/> </p>

<p>Boxplot (isoforms, replicates)</p>

<p><img src="figure/boxplot_isoforms_replicates.png" alt="plot of chunk boxplot_isoforms_replicates"/> </p>

<h3>Density</h3>

<p>Density (genes)</p>

<p><img src="figure/density.png" alt="plot of chunk density"/> </p>

<p>Density (genes, replicates)</p>

<p><img src="figure/density_w_replicates.png" alt="plot of chunk density_w_replicates"/> </p>

<h2>Clustering</h2>

<h3>Replicate Clusters</h3>

<p><img src="figure/replicate_clusters.png" alt="plot of chunk replicate_clusters"/> </p>

<pre><code>## &#39;dendrogram&#39; with 2 branches and 17 members total, at height 0.07545
</code></pre>

<h3>PCA (genes)</h3>

<p><img src="figure/PCA.png" alt="plot of chunk PCA"/> </p>

<h3>MDS (genes)</h3>

<p><img src="figure/MDS.png" alt="plot of chunk MDS"/> </p>

<h1>KO assessment</h1>

<h2>Endogenous lncRNA expression</h2>

<p><img src="figure/Enodenous_lncRNA_tables.png" alt="plot of chunk Enodenous_lncRNA_tables"/> </p>

<p>Endogenous expression of Celrr isoforms:</p>

<p><img src="figure/endogenous_iso.png" alt="plot of chunk endogenous_iso"/> </p>

<p>Barplot of gene expression:</p>

<p><img src="figure/endogenous_barplot.png" alt="plot of chunk endogenous_barplot"/> </p>

<p>Barplot of isoform expression:</p>

<p><img src="figure/endogenous_iso_barplot.png" alt="plot of chunk endogenous_iso_barplot"/> </p>

<h2>LacZ expression</h2>

<p><img src="figure/LacZ_expression1.png" alt="plot of chunk LacZ_expression"/> <img src="figure/LacZ_expression2.png" alt="plot of chunk LacZ_expression"/> </p>

<h2>Digital Genotyping (LacZ vs Endogenous lncRNA and Sex)</h2>

<p>Expression plot (endogenous linc, lacZ, Y-expressed gene):</p>

<p><img src="figure/Digital_Genotyping.png" alt="plot of chunk Digital_Genotyping"/> </p>

<p>Expression heatmap:
<img src="figure/digital_geno_heatmap.png" alt="plot of chunk digital_geno_heatmap"/> </p>

<h1>Differential Analysis</h1>

<h2>Differential Genes</h2>

<p>There are 52 significantly differentially expressed genes. They are:</p>

<!-- html table generated in R 3.0.2 by xtable 1.7-3 package -->

<!-- Mon Jun 23 17:19:38 2014 -->

<TABLE border=1>
<TR> <TH>  </TH> <TH> geneAnnot$gene_short_name </TH>  </TR>
  <TR> <TD align="right"> 1 </TD> <TD> Chordc1 </TD> </TR>
  <TR> <TD align="right"> 2 </TD> <TD> Nes </TD> </TR>
  <TR> <TD align="right"> 3 </TD> <TD> Dnajb1 </TD> </TR>
  <TR> <TD align="right"> 4 </TD> <TD> Dio2 </TD> </TR>
  <TR> <TD align="right"> 5 </TD> <TD> Slc38a3 </TD> </TR>
  <TR> <TD align="right"> 6 </TD> <TD> P4ha1 </TD> </TR>
  <TR> <TD align="right"> 7 </TD> <TD> Btg2 </TD> </TR>
  <TR> <TD align="right"> 8 </TD> <TD> Ccdc117 </TD> </TR>
  <TR> <TD align="right"> 9 </TD> <TD> Pdia6 </TD> </TR>
  <TR> <TD align="right"> 10 </TD> <TD> Adi1 </TD> </TR>
  <TR> <TD align="right"> 11 </TD> <TD> Serpina3n </TD> </TR>
  <TR> <TD align="right"> 12 </TD> <TD> Fos </TD> </TR>
  <TR> <TD align="right"> 13 </TD> <TD> Arc </TD> </TR>
  <TR> <TD align="right"> 14 </TD> <TD> Sdf2l1 </TD> </TR>
  <TR> <TD align="right"> 15 </TD> <TD> Nr4a1 </TD> </TR>
  <TR> <TD align="right"> 16 </TD> <TD> Creld2 </TD> </TR>
  <TR> <TD align="right"> 17 </TD> <TD> Glo1 </TD> </TR>
  <TR> <TD align="right"> 18 </TD> <TD> Dusp1 </TD> </TR>
  <TR> <TD align="right"> 19 </TD> <TD> Pdia4 </TD> </TR>
  <TR> <TD align="right"> 20 </TD> <TD> Lcn2 </TD> </TR>
  <TR> <TD align="right"> 21 </TD> <TD> Hspa5 </TD> </TR>
  <TR> <TD align="right"> 22 </TD> <TD> Vcam1 </TD> </TR>
  <TR> <TD align="right"> 23 </TD> <TD> Hddc3 </TD> </TR>
  <TR> <TD align="right"> 24 </TD> <TD> Tsc22d3 </TD> </TR>
  <TR> <TD align="right"> 25 </TD> <TD> Mt2 </TD> </TR>
  <TR> <TD align="right"> 26 </TD> <TD> Mt1 </TD> </TR>
  <TR> <TD align="right"> 27 </TD> <TD> Pllp </TD> </TR>
  <TR> <TD align="right"> 28 </TD> <TD> Cryab </TD> </TR>
  <TR> <TD align="right"> 29 </TD> <TD> Ptgs2 </TD> </TR>
  <TR> <TD align="right"> 30 </TD> <TD> Lars2 </TD> </TR>
  <TR> <TD align="right"> 31 </TD> <TD> Egr2 </TD> </TR>
  <TR> <TD align="right"> 32 </TD> <TD> Fmo2 </TD> </TR>
  <TR> <TD align="right"> 33 </TD> <TD> Obfc1 </TD> </TR>
  <TR> <TD align="right"> 34 </TD> <TD> Pcsk9 </TD> </TR>
  <TR> <TD align="right"> 35 </TD> <TD> Gm9493 </TD> </TR>
  <TR> <TD align="right"> 36 </TD> <TD> Acp1 </TD> </TR>
  <TR> <TD align="right"> 37 </TD> <TD> Cirbp </TD> </TR>
  <TR> <TD align="right"> 38 </TD> <TD> Npas4 </TD> </TR>
  <TR> <TD align="right"> 39 </TD> <TD> Tmem252 </TD> </TR>
  <TR> <TD align="right"> 40 </TD> <TD> Opalin </TD> </TR>
  <TR> <TD align="right"> 41 </TD> <TD> Cd14 </TD> </TR>
  <TR> <TD align="right"> 42 </TD> <TD> Rpl30 </TD> </TR>
  <TR> <TD align="right"> 43 </TD> <TD> Tpt1 </TD> </TR>
  <TR> <TD align="right"> 44 </TD> <TD> Zbtb40 </TD> </TR>
  <TR> <TD align="right"> 45 </TD> <TD> Rpl34 </TD> </TR>
  <TR> <TD align="right"> 46 </TD> <TD> Ppia </TD> </TR>
  <TR> <TD align="right"> 47 </TD> <TD> Gm7292 </TD> </TR>
  <TR> <TD align="right"> 48 </TD> <TD> H2-Bl </TD> </TR>
  <TR> <TD align="right"> 49 </TD> <TD> Wdfy1 </TD> </TR>
  <TR> <TD align="right"> 50 </TD> <TD> Hspa1b </TD> </TR>
  <TR> <TD align="right"> 51 </TD> <TD> Hspa1a </TD> </TR>
  <TR> <TD align="right"> 52 </TD> <TD> AA465934 </TD> </TR>
   </TABLE>

<h3>Matrix of gene significant differences between conditions</h3>

<p>(skip for Brainmap wt-v-ko comparisons)</p>

<p><img src="figure/sigMatrix.png" alt="plot of chunk sigMatrix"/> </p>

<h3>Significant gene expression differences between conditions</h3>

<p>Expression plot (genes):
<img src="figure/sigExpression1.png" alt="plot of chunk sigExpression"/> <img src="figure/sigExpression2.png" alt="plot of chunk sigExpression"/> </p>

<p>Significant genes with expression &gt;50fpkm (any condition):
<img src="figure/highly_expressed_sig.png" alt="plot of chunk highly_expressed_sig"/> </p>

<p>An individual look at each of the highly expressed significantly differentially regulated genes:
(eval=false for first pass)</p>

<h3>Expression-level/significance relationship</h3>

<p>Scatter plot of significant genes only:</p>

<pre><code>## Error: One or more values of &#39;x&#39; or &#39;y&#39; are not valid sample names!
</code></pre>

<p>Volcano plot with significant genes only:</p>

<pre><code>## Error: One or more values of &#39;x&#39; or &#39;y&#39; are not valid sample names!
</code></pre>

<h2>Differential Splicing</h2>

<h3>Differential Isoforms between conditions</h3>

<p>Per isoform difference between conditions:
<img src="figure/diff_iso.png" alt="plot of chunk diff_iso"/> </p>

<pre><code>## Error: RS-DBI driver: (error in statement: near &quot;)&quot;: syntax error)
</code></pre>

<p>These isoforms are:</p>

<pre><code>## Error: NaN dissimilarity value.
</code></pre>

<h3>Differential Splicing between conditions</h3>

<p>(eval false for first pass)</p>

<p>Per condition differences in isoforms (Does gene have diff piechart between conditions?)</p>

<p>These genes are:</p>

<p>Splicing heatmap by isoform:</p>

<p>Splicing heatmap by gene</p>

<p>The following are significantly differentially spliced genes (relative portion of isoform per condition): </p>

<h1>Gene/Pathway Analysis</h1>

<h2>GSEA</h2>

<p><img src="figure/print_GSEA_biocarta.png" alt="plot of chunk print_GSEA_biocarta"/> </p>

<p><img src="figure/print_GSEA_reactome.png" alt="plot of chunk print_GSEA_reactome"/> </p>

<h2>GO enrichment</h2>

<p>Cluster profiler used to call enichments of significantly differentially regulated genes that map to Entrez IDs. </p>

<pre><code>## [1] &quot;ERROR: The estimated pi0 &lt;= 0. Check that you have valid p-values or use another lambda method.&quot;
</code></pre>

<pre><code>## [1] &quot;ERROR: The estimated pi0 &lt;= 0. Check that you have valid p-values or use another lambda method.&quot;
</code></pre>

<p><img src="figure/GO_figures1.png" alt="plot of chunk GO_figures"/> <img src="figure/GO_figures2.png" alt="plot of chunk GO_figures"/> <img src="figure/GO_figures3.png" alt="plot of chunk GO_figures"/> <img src="figure/GO_figures4.png" alt="plot of chunk GO_figures"/> <img src="figure/GO_figures5.png" alt="plot of chunk GO_figures"/> </p>

<h1>Cis vs Trans (locally)</h1>

<pre><code>## Error: argument &quot;seed&quot; is missing, with no default
</code></pre>

<p>The pvalue for 0 genes significantly regulated in this region is: 1 </p>

<p><img src="figure/overlap_image.png" alt="plot of chunk overlap_image"/> </p>

<h1>Notes</h1>

<h2>Samples used are:</h2>

<!-- html table generated in R 3.0.2 by xtable 1.7-3 package -->

<!-- Mon Jun 23 17:42:43 2014 -->

<TABLE border=1>
<TR> <TH>  </TH> <TH> 10 </TH>  </TR>
  <TR> <TD align="right"> 1 </TD> <TD> JR729 </TD> </TR>
  <TR> <TD align="right"> 2 </TD> <TD> JR728 </TD> </TR>
  <TR> <TD align="right"> 3 </TD> <TD> JR796 </TD> </TR>
  <TR> <TD align="right"> 4 </TD> <TD> JR797 </TD> </TR>
  <TR> <TD align="right"> 5 </TD> <TD> JR740 </TD> </TR>
  <TR> <TD align="right"> 6 </TD> <TD> JR800 </TD> </TR>
  <TR> <TD align="right"> 7 </TD> <TD> JR827 </TD> </TR>
  <TR> <TD align="right"> 8 </TD> <TD> JR778 </TD> </TR>
  <TR> <TD align="right"> 9 </TD> <TD> JR734 </TD> </TR>
  <TR> <TD align="right"> 10 </TD> <TD> JR802 </TD> </TR>
  <TR> <TD align="right"> 11 </TD> <TD> JR803 </TD> </TR>
  <TR> <TD align="right"> 12 </TD> <TD> JR735 </TD> </TR>
  <TR> <TD align="right"> 13 </TD> <TD> JR785 </TD> </TR>
  <TR> <TD align="right"> 14 </TD> <TD> JR781 </TD> </TR>
  <TR> <TD align="right"> 15 </TD> <TD> JR739 </TD> </TR>
  <TR> <TD align="right"> 16 </TD> <TD> JR783 </TD> </TR>
  <TR> <TD align="right"> 17 </TD> <TD> JR804 </TD> </TR>
   </TABLE>

<h2>Replicates</h2>

<!-- html table generated in R 3.0.2 by xtable 1.7-3 package -->

<!-- Mon Jun 23 17:42:43 2014 -->

<TABLE border=1>
<TR> <TH>  </TH> <TH> file </TH> <TH> sample_name </TH> <TH> replicate </TH> <TH> rep_name </TH> <TH> total_mass </TH> <TH> norm_mass </TH> <TH> internal_scale </TH> <TH> external_scale </TH>  </TR>
  <TR> <TD align="right"> 1 </TD> <TD> /n/rinn_data1/seq/lgoff/Projects/BrainMap/data/quants/JR729/abundances.cxb </TD> <TD> WT </TD> <TD align="right">   0 </TD> <TD> WT_0 </TD> <TD align="right"> 26338300.00 </TD> <TD align="right"> 33873900.00 </TD> <TD align="right"> 0.78 </TD> <TD align="right"> 1.00 </TD> </TR>
  <TR> <TD align="right"> 2 </TD> <TD> /n/rinn_data1/seq/lgoff/Projects/BrainMap/data/quants/JR728/abundances.cxb </TD> <TD> WT </TD> <TD align="right">   1 </TD> <TD> WT_1 </TD> <TD align="right"> 20332100.00 </TD> <TD align="right"> 33873900.00 </TD> <TD align="right"> 0.60 </TD> <TD align="right"> 1.00 </TD> </TR>
  <TR> <TD align="right"> 3 </TD> <TD> /n/rinn_data1/seq/lgoff/Projects/BrainMap/data/quants/JR796/abundances.cxb </TD> <TD> WT </TD> <TD align="right">   2 </TD> <TD> WT_2 </TD> <TD align="right"> 34093300.00 </TD> <TD align="right"> 33873900.00 </TD> <TD align="right"> 1.00 </TD> <TD align="right"> 1.00 </TD> </TR>
  <TR> <TD align="right"> 4 </TD> <TD> /n/rinn_data1/seq/lgoff/Projects/BrainMap/data/quants/JR797/abundances.cxb </TD> <TD> WT </TD> <TD align="right">   3 </TD> <TD> WT_3 </TD> <TD align="right"> 28111900.00 </TD> <TD align="right"> 33873900.00 </TD> <TD align="right"> 0.82 </TD> <TD align="right"> 1.00 </TD> </TR>
  <TR> <TD align="right"> 5 </TD> <TD> /n/rinn_data1/seq/lgoff/Projects/BrainMap/data/quants/JR740/abundances.cxb </TD> <TD> WT </TD> <TD align="right">   4 </TD> <TD> WT_4 </TD> <TD align="right"> 35813300.00 </TD> <TD align="right"> 33873900.00 </TD> <TD align="right"> 1.06 </TD> <TD align="right"> 1.00 </TD> </TR>
  <TR> <TD align="right"> 6 </TD> <TD> /n/rinn_data1/seq/lgoff/Projects/BrainMap/data/quants/JR800/abundances.cxb </TD> <TD> WT </TD> <TD align="right">   5 </TD> <TD> WT_5 </TD> <TD align="right"> 37017800.00 </TD> <TD align="right"> 33873900.00 </TD> <TD align="right"> 1.10 </TD> <TD align="right"> 1.00 </TD> </TR>
  <TR> <TD align="right"> 7 </TD> <TD> /n/rinn_data1/seq/lgoff/Projects/BrainMap/data/quants/JR827/abundances.cxb </TD> <TD> WT </TD> <TD align="right">   6 </TD> <TD> WT_6 </TD> <TD align="right"> 27790600.00 </TD> <TD align="right"> 33873900.00 </TD> <TD align="right"> 0.82 </TD> <TD align="right"> 1.00 </TD> </TR>
  <TR> <TD align="right"> 8 </TD> <TD> /n/rinn_data1/seq/lgoff/Projects/BrainMap/data/quants/JR778/abundances.cxb </TD> <TD> WT </TD> <TD align="right">   7 </TD> <TD> WT_7 </TD> <TD align="right"> 39549100.00 </TD> <TD align="right"> 33873900.00 </TD> <TD align="right"> 1.18 </TD> <TD align="right"> 1.00 </TD> </TR>
  <TR> <TD align="right"> 9 </TD> <TD> /n/rinn_data1/seq/lgoff/Projects/BrainMap/data/quants/JR734/abundances.cxb </TD> <TD> WT </TD> <TD align="right">   8 </TD> <TD> WT_8 </TD> <TD align="right"> 34485000.00 </TD> <TD align="right"> 33873900.00 </TD> <TD align="right"> 1.03 </TD> <TD align="right"> 1.00 </TD> </TR>
  <TR> <TD align="right"> 10 </TD> <TD> /n/rinn_data1/seq/lgoff/Projects/BrainMap/data/quants/JR802/abundances.cxb </TD> <TD> WT </TD> <TD align="right">   9 </TD> <TD> WT_9 </TD> <TD align="right"> 45472800.00 </TD> <TD align="right"> 33873900.00 </TD> <TD align="right"> 1.33 </TD> <TD align="right"> 1.00 </TD> </TR>
  <TR> <TD align="right"> 11 </TD> <TD> /n/rinn_data1/seq/lgoff/Projects/BrainMap/data/quants/JR803/abundances.cxb </TD> <TD> WT </TD> <TD align="right">  10 </TD> <TD> WT_10 </TD> <TD align="right"> 52136700.00 </TD> <TD align="right"> 33873900.00 </TD> <TD align="right"> 1.54 </TD> <TD align="right"> 1.00 </TD> </TR>
  <TR> <TD align="right"> 12 </TD> <TD> /n/rinn_data1/seq/lgoff/Projects/BrainMap/data/quants/JR735/abundances.cxb </TD> <TD> WT </TD> <TD align="right">  11 </TD> <TD> WT_11 </TD> <TD align="right"> 34999600.00 </TD> <TD align="right"> 33873900.00 </TD> <TD align="right"> 1.04 </TD> <TD align="right"> 1.00 </TD> </TR>
  <TR> <TD align="right"> 13 </TD> <TD> /n/rinn_data1/seq/lgoff/Projects/BrainMap/data/quants/JR785/abundances.cxb </TD> <TD> WT </TD> <TD align="right">  12 </TD> <TD> WT_12 </TD> <TD align="right"> 34180700.00 </TD> <TD align="right"> 33873900.00 </TD> <TD align="right"> 1.00 </TD> <TD align="right"> 1.00 </TD> </TR>
  <TR> <TD align="right"> 14 </TD> <TD> /n/rinn_data1/seq/lgoff/Projects/BrainMap/data/quants/JR781/abundances.cxb </TD> <TD> WT </TD> <TD align="right">  13 </TD> <TD> WT_13 </TD> <TD align="right"> 41544600.00 </TD> <TD align="right"> 33873900.00 </TD> <TD align="right"> 1.23 </TD> <TD align="right"> 1.00 </TD> </TR>
  <TR> <TD align="right"> 15 </TD> <TD> /n/rinn_data1/seq/lgoff/Projects/BrainMap/data/quants/JR739/abundances.cxb </TD> <TD> Celrr </TD> <TD align="right">   0 </TD> <TD> Celrr_0 </TD> <TD align="right"> 23329000.00 </TD> <TD align="right"> 33873900.00 </TD> <TD align="right"> 0.69 </TD> <TD align="right"> 1.00 </TD> </TR>
  <TR> <TD align="right"> 16 </TD> <TD> /n/rinn_data1/seq/lgoff/Projects/BrainMap/data/quants/JR783/abundances.cxb </TD> <TD> Celrr </TD> <TD align="right">   1 </TD> <TD> Celrr_1 </TD> <TD align="right"> 37164200.00 </TD> <TD align="right"> 33873900.00 </TD> <TD align="right"> 1.10 </TD> <TD align="right"> 1.00 </TD> </TR>
  <TR> <TD align="right"> 17 </TD> <TD> /n/rinn_data1/seq/lgoff/Projects/BrainMap/data/quants/JR804/abundances.cxb </TD> <TD> Celrr </TD> <TD align="right">   2 </TD> <TD> Celrr_2 </TD> <TD align="right"> 39723200.00 </TD> <TD align="right"> 33873900.00 </TD> <TD align="right"> 1.16 </TD> <TD align="right"> 1.00 </TD> </TR>
   </TABLE>

<h2>Session Info</h2>

<pre><code>## R version 3.0.2 (2013-09-25)
## Platform: x86_64-unknown-linux-gnu (64-bit)
## 
## locale:
##  [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
##  [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
##  [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
##  [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
##  [9] LC_ADDRESS=C               LC_TELEPHONE=C            
## [11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       
## 
## attached base packages:
## [1] grid      parallel  stats     graphics  grDevices utils     datasets 
## [8] methods   base     
## 
## other attached packages:
##  [1] plyr_1.8.1                         
##  [2] stringr_0.6.2                      
##  [3] seqbias_1.10.0                     
##  [4] BSgenome.Mmusculus.UCSC.mm10_1.3.19
##  [5] BSgenome_1.30.0                    
##  [6] Biostrings_2.30.1                  
##  [7] GO.db_2.10.1                       
##  [8] org.Mm.eg.db_2.10.1                
##  [9] clusterProfiler_1.13.1             
## [10] DOSE_2.0.0                         
## [11] ReactomePA_1.6.1                   
## [12] AnnotationDbi_1.24.0               
## [13] Biobase_2.22.0                     
## [14] mgcv_1.7-29                        
## [15] nlme_3.1-117                       
## [16] gridExtra_0.9.1                    
## [17] gtable_0.1.2                       
## [18] marray_1.40.0                      
## [19] gplots_2.13.0                      
## [20] GSA_1.03                           
## [21] limma_3.18.13                      
## [22] xtable_1.7-3                       
## [23] cummeRbund_2.7.2                   
## [24] Gviz_1.6.0                         
## [25] rtracklayer_1.22.7                 
## [26] GenomicRanges_1.14.4               
## [27] XVector_0.2.0                      
## [28] IRanges_1.20.7                     
## [29] fastcluster_1.1.13                 
## [30] reshape2_1.4                       
## [31] ggplot2_1.0.0                      
## [32] RSQLite_0.11.4                     
## [33] DBI_0.2-7                          
## [34] BiocGenerics_0.8.0                 
## [35] knitr_1.6                          
## 
## loaded via a namespace (and not attached):
##  [1] biomaRt_2.18.0         biovizBase_1.10.8      bitops_1.0-6          
##  [4] caTools_1.17           cluster_1.15.2         codetools_0.2-8       
##  [7] colorspace_1.2-4       dichromat_2.0-0        digest_0.6.4          
## [10] DO.db_2.7              evaluate_0.5.5         formatR_0.10          
## [13] Formula_1.1-1          gdata_2.13.3           GenomicFeatures_1.14.5
## [16] GOSemSim_1.20.3        graph_1.40.1           graphite_1.8.1        
## [19] gtools_3.4.1           Hmisc_3.14-4           igraph_0.7.1          
## [22] KEGG.db_2.10.1         KernSmooth_2.23-12     labeling_0.2          
## [25] lattice_0.20-29        latticeExtra_0.6-26    MASS_7.3-33           
## [28] Matrix_1.1-3           munsell_0.4.2          org.Hs.eg.db_2.10.1   
## [31] proto_0.3-10           qvalue_1.36.0          RColorBrewer_1.0-5    
## [34] Rcpp_0.11.1            RCurl_1.95-4.1         reactome.db_1.46.1    
## [37] Rsamtools_1.14.3       scales_0.2.4           splines_3.0.2         
## [40] stats4_3.0.2           survival_2.37-7        tcltk_3.0.2           
## [43] tools_3.0.2            XML_3.98-1.1           zlibbioc_1.8.0
</code></pre>

<h2>Run Info</h2>

<pre><code>##           param
## 1      cmd_line
## 2       version
## 3  SVN_revision
## 4 boost_version
## 5        genome
##                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   value
## 1 cuffdiff -p 10 -L WT,Celrr -o /n/rinn_data1/seq/lgoff/Projects/BrainMap/data/diffs/Celrr_vs_WT_Adult /n/rinn_data1/seq/lgoff/Projects/BrainMap/data/annotation/mm10_gencode_vM2_with_lncRNAs_and_LacZ.gtf /n/rinn_data1/seq/lgoff/Projects/BrainMap/data/quants/JR729/abundances.cxb,/n/rinn_data1/seq/lgoff/Projects/BrainMap/data/quants/JR728/abundances.cxb,/n/rinn_data1/seq/lgoff/Projects/BrainMap/data/quants/JR796/abundances.cxb,/n/rinn_data1/seq/lgoff/Projects/BrainMap/data/quants/JR797/abundances.cxb,/n/rinn_data1/seq/lgoff/Projects/BrainMap/data/quants/JR740/abundances.cxb,/n/rinn_data1/seq/lgoff/Projects/BrainMap/data/quants/JR800/abundances.cxb,/n/rinn_data1/seq/lgoff/Projects/BrainMap/data/quants/JR827/abundances.cxb,/n/rinn_data1/seq/lgoff/Projects/BrainMap/data/quants/JR778/abundances.cxb,/n/rinn_data1/seq/lgoff/Projects/BrainMap/data/quants/JR734/abundances.cxb,/n/rinn_data1/seq/lgoff/Projects/BrainMap/data/quants/JR802/abundances.cxb,/n/rinn_data1/seq/lgoff/Projects/BrainMap/data/quants/JR803/abundances.cxb,/n/rinn_data1/seq/lgoff/Projects/BrainMap/data/quants/JR735/abundances.cxb,/n/rinn_data1/seq/lgoff/Projects/BrainMap/data/quants/JR785/abundances.cxb,/n/rinn_data1/seq/lgoff/Projects/BrainMap/data/quants/JR781/abundances.cxb /n/rinn_data1/seq/lgoff/Projects/BrainMap/data/quants/JR739/abundances.cxb,/n/rinn_data1/seq/lgoff/Projects/BrainMap/data/quants/JR783/abundances.cxb,/n/rinn_data1/seq/lgoff/Projects/BrainMap/data/quants/JR804/abundances.cxb 
## 2                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 2.2.1
## 3                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  4237
## 4                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                104700
## 5                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  mm10
</code></pre>

</body>

</html>
