Master Script for PNAS BrainMap Study
=====================================

#Setup and Imports
```{r setwd}
basedir = "/Volumes/Odyssey/seq/lgoff/Projects/BrainMap/analysis"
#basedir = "/n/rinn_data1/seq/lgoff/Projects/BrainMap/analysis"
setwd(basedir)
```

```{r init}
library(cummeRbund)
library(ggplot2)
```

## Initialize cuffData.db and establish connection
```{r cuff}
refGTF <- "../data/annotations/mm9_ucsc_and_lincs_DB2.0.gtf"
cuffdir <- "../data/diffs/fulldiff"

cuff<-readCufflinks(cuffdif,gtfFile=refGTF,genome="mm9")

samples(cuff)

replicates(cuff)
```

## Quality Control

## Overdispersion fit
```{r QC_dispersion,fig.width=20,fig.height=20}
dispersionPlot(cuff)
```


### Dendrogram
```{r QC_dendro,fig.width=20,fig.height=10}
csDendro(genes(cuff),rep=T)
```

### Squared Coefficient of Variation
Evaluates cross-replicate variability across expression regime for gene- and isoform-level expression estimates per condition.
```{r QC_SCV,fig.width=10,fig.height=10,cache=TRUE}
fpkmSCVPlot(genes(cuff))

fpkmSCVPlot(isoforms(cuff))
```

### Dimensionality reduction

```{r QC_dimRedux,fig.width=20,fig.height=20}
MDSplot(genes(cuff))

PCAplot(genes(cuff),rep=TRUE)
```

## Significant gene expression differences (global)
```{r sigGenes}
alpha<-0.001
sigGenes<-getSig(cuff,alpha=alpha)
```


## Global Analysis
```{r global_heatmap,fig.width=15,fig.height=30}
csHeatmap(sigGenes,rep=T)
```


## GSEA?

## Guilt-by-association on FC values?

## Session Info
```{r sessionInfo}
sessionInfo()
```


