Mouse Data Overview 1
========================================================
```{r options}
opts_chunk$set(echo=FALSE)
```

```{r setup}
library(cummeRbund)

analysisdir<-"/n/rinn_data1/users/agroff/GITHUB/BrainMap/analysis/"
diffdir<-"/n/rinn_data1/seq/lgoff/Projects/BrainMap/data/diffs"
GTF<-"/n/rinn_data1/seq/lgoff/Projects/BrainMap/data/annotation/mm10_gencode_vM2_with_lncRNAs_and_LacZ.gtf"

dir<-"/n/rinn_data1/seq/lgoff/Projects/BrainMap/data/diffs/Celrr_vs_WT_Adult"
setwd(dir)
cuff<-readCufflinks(gtfFile=GTF,genome="mm10")
```

# Adult Samples 

- all have strange density plots: add pseudocount 

Without Pseudocount
```{r density}
csDensity(genes(cuff),replicates=TRUE)
```


With Pseudocount = 0.01 
```{r density pseudo}
csDensity(genes(cuff),pseudocount=0.01,replicates=TRUE)
```


## WTS:

### Genotyping
- wt 12 is ko/het (JR817)
```{r genotyping}
genotypingGeneIDs<-c(strain,"LacZ","Eif2s3y")
genotypingGenes<-getGenes(cuff,genotypingGeneIDs)
csHeatmap(genotypingGenes,replicates=T)
```

### QC 
- wts 9, 10 have smaller fpkm distribution (replicates)

```{r boxplot replicates}
csBoxplot(genes(cuff),replicates=T)
```

- isoform fpkm distributions: WT smaller on whole, but replicates look fine? 

Boxplot (isoforms)

```{r boxplot isoforms}
csBoxplot(isoforms(cuff))
```

Boxplot (isoforms, replicates)

```{r boxplot isoforms replicates}
csBoxplot(isoforms(cuff), replicates=T)
```


### Other Notes

## KO Strains

### Celrr

### Crnde

- Either wrong genotyping or isoform beyond KO region stabilized by LZ (need to see tracks)

```{r crnde adult}
dir<-"/n/rinn_data1/seq/lgoff/Projects/BrainMap/data/diffs/Crnde_vs_WT_Adult"
setwd(dir)
cuff<-readCufflinks()
id<-"Crnde"
myGene<-getGene(cuff,id)
expressionBarplot(isoforms(myGene), replicates=T)
```


- Wrong genotyping CRNDE_2 (JR818)

```{r crnde lz adult}
id<-"LacZ"
myGene<-getGene(cuff,id)
expressionBarplot(isoforms(myGene), replicates=T)
ids<-c("Crnde","LacZ")
myGenes<-getGenes(cuff,ids)
csHeatmap(myGenes,replicates=TRUE)
```

### Eldr

- Either wrong genotyping or isoform beyond KO region stabilized by LZ (need to see tracks)

```{r eldr adult}
dir<-"/n/rinn_data1/seq/lgoff/Projects/BrainMap/data/diffs/Eldr_vs_WT_Adult"
setwd(dir)
cuff<-readCufflinks()
id<-"Eldr"
myGene<-getGene(cuff,id)
expressionBarplot(isoforms(myGene), replicates=T)

```


```{r eldr lz adult}
id<-"LacZ"
myGene<-getGene(cuff,id)
expressionBarplot(isoforms(myGene), replicates=T)
ids<-c("Eldr","LacZ")
myGenes<-getGenes(cuff,ids)
csHeatmap(myGenes,replicates=TRUE)
```

Problem child is Eldr_0 (JR733)
Does Eldr_1 look like a het? (JR799)

### Haunt

### Kantr (Jarid1C)
YAY! They actually cluster!
```{r Kantr adult}
dir<-"/n/rinn_data1/seq/lgoff/Projects/BrainMap/data/diffs/Kantr_vs_WT_Adult"
setwd(dir)
cuff<-readCufflinks()

csDendro(genes(cuff),replicates=TRUE)

id<-"Kantr"
myGene<-getGene(cuff,id)

```

### Brn1a
3 isoforms expressed

### Brn1b
LacZ expressed slightly higher than lincRNA

### linc-Cox2
YAY! They cluster together! (is this a litter thing? sex difference thing?)

```{r lincCox2 adult}
dir<-"/n/rinn_data1/seq/lgoff/Projects/BrainMap/data/diffs/linc-Cox2_vs_WT_Adult"
setwd(dir)
cuff<-readCufflinks()

csDendro(genes(cuff),replicates=TRUE)

id<-"linc-Cox2"
myGene<-getGene(cuff,id)

```

**THIS DENDROGRAM LOOKS DIFFERENT WHEN PRINTED IN R STUDIO!**

Endogenous linc and LZ approx same expression levels 


### linc-Enc1a

I think this one MIGHT be a mapping error. Endogenous linc expressed at ~ WT levels (very low), but all cells have lacZ robustly expressed. Potentially hets I guess 

```{r lincEnc1a}
dir<-"/n/rinn_data1/seq/lgoff/Projects/BrainMap/data/diffs/linc-Enc1_vs_WT_Adult"
setwd(dir)
cuff<-readCufflinks()
id<-"linc-Enc1"
myGene<-getGene(cuff,id)
expressionBarplot(isoforms(myGene), replicates=T)
```

LacZ and genotyping heatmap
```{r lincEnc1a lz adult}
id<-"LacZ"
myGene<-getGene(cuff,id)
expressionBarplot(isoforms(myGene), replicates=T)
ids<-c("linc-Enc1","LacZ")
myGenes<-getGenes(cuff,ids)
csHeatmap(myGenes,replicates=TRUE)
```


### Manr
Same as with linc-Enc1a, may be extra-knockout region OR perhaps hets or some sort of artifact.. 

```{r MANR}
dir<-"/n/rinn_data1/seq/lgoff/Projects/BrainMap/data/diffs/Manr_vs_WT_Adult"
setwd(dir)
cuff<-readCufflinks()
id<-"Manr"
myGene<-getGene(cuff,id)
expressionBarplot(isoforms(myGene), replicates=T)
```

LacZ and genotyping heatmap
```{r MANR lz adult}
lz<-"LacZ"
myGene<-getGene(cuff,lz)
expressionBarplot(isoforms(myGene), replicates=T)
ids<-c(id,lz)
myGenes<-getGenes(cuff,ids)
csHeatmap(myGenes,replicates=TRUE)
```


### Peril

Fairly certai this is extra-knockout region. Need to make tracks! 

```{r Peril}
dir<-"/n/rinn_data1/seq/lgoff/Projects/BrainMap/data/diffs/Peril_vs_WT_Adult"
setwd(dir)
cuff<-readCufflinks()
id<-"Peril"
myGene<-getGene(cuff,id)
expressionBarplot(isoforms(myGene), replicates=T)
```

LacZ and genotyping heatmap
```{r Peril lz adult}
lz<-"LacZ"
myGene<-getGene(cuff,lz)
expressionBarplot(isoforms(myGene), replicates=T)
ids<-c(id,lz)
myGenes<-getGenes(cuff,ids)
csHeatmap(myGenes,replicates=TRUE)
```

### Trp53cor1 (linc-p21)


### Tug1

LacZ expressed far lower than Tug1 lincRNA 
Why is one of the tug1 replicates missing from plots/dendrogram??



# Embryonic Samples
- Celrr: celrr 0 is not KO (seems WT)
- wt13 

## WTS:

### Genotyping

### QC 

### Other Notes

## KO Strains

### Celrr

### Crnde

### Eldr

### Haunt

### Kantr

### Brn1a

### Brn1b

### linc-Cox2

### linc-Enc1a

### Manr

### Peril

### Trp53cor (linc-p21)

### Tug1