<!DOCTYPE html>
<!-- saved from url=(0014)about:internet -->
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>

<title>Mouse Data Overview DE genelists </title>

<style type="text/css">
body, td {
   font-family: sans-serif;
   background-color: white;
   font-size: 12px;
   margin: 8px;
}

tt, code, pre {
   font-family: 'DejaVu Sans Mono', 'Droid Sans Mono', 'Lucida Console', Consolas, Monaco, monospace;
}

h1 { 
   font-size:2.2em; 
}

h2 { 
   font-size:1.8em; 
}

h3 { 
   font-size:1.4em; 
}

h4 { 
   font-size:1.0em; 
}

h5 { 
   font-size:0.9em; 
}

h6 { 
   font-size:0.8em; 
}

a:visited {
   color: rgb(50%, 0%, 50%);
}

pre {	
   margin-top: 0;
   max-width: 95%;
   border: 1px solid #ccc;
   white-space: pre-wrap;
}

pre code {
   display: block; padding: 0.5em;
}

code.r, code.cpp {
   background-color: #F8F8F8;
}

table, td, th {
  border: none;
}

blockquote {
   color:#666666;
   margin:0;
   padding-left: 1em;
   border-left: 0.5em #EEE solid;
}

hr {
   height: 0px;
   border-bottom: none;
   border-top-width: thin;
   border-top-style: dotted;
   border-top-color: #999999;
}

@media print {
   * { 
      background: transparent !important; 
      color: black !important; 
      filter:none !important; 
      -ms-filter: none !important; 
   }

   body { 
      font-size:12pt; 
      max-width:100%; 
   }
       
   a, a:visited { 
      text-decoration: underline; 
   }

   hr { 
      visibility: hidden;
      page-break-before: always;
   }

   pre, blockquote { 
      padding-right: 1em; 
      page-break-inside: avoid; 
   }

   tr, img { 
      page-break-inside: avoid; 
   }

   img { 
      max-width: 100% !important; 
   }

   @page :left { 
      margin: 15mm 20mm 15mm 10mm; 
   }
     
   @page :right { 
      margin: 15mm 10mm 15mm 20mm; 
   }

   p, h2, h3 { 
      orphans: 3; widows: 3; 
   }

   h2, h3 { 
      page-break-after: avoid; 
   }
}

</style>





</head>

<body>
<h1>Mouse Data Overview DE genelists </h1>

<h1>ADULT KNOCKOUT STRAINS</h1>

<h3>Celrr</h3>

<p>Number of differentially expressed genes:54</p>

<h3>Crnde</h3>

<p>Number of differentially expressed genes:42</p>

<h3>Eldr</h3>

<p>Number of differentially expressed genes:27</p>

<h3>Haunt</h3>

<p>Number of differentially expressed genes:37</p>

<h3>Kantr (Jarid1C)</h3>

<p>Number of differentially expressed genes:315</p>

<h3>Brn1a</h3>

<p>3 isoforms expressed</p>

<p>Number of differentially expressed genes:74</p>

<h3>Brn1b</h3>

<p>LacZ expressed slightly higher than lincRNA</p>

<p>Number of differentially expressed genes:193</p>

<h3>linc-Cox2</h3>

<p>Endogenous linc and LZ approx same expression levels </p>

<p>Number of differentially expressed genes:94</p>

<h3>linc-Enc1a</h3>

<p>Number of differentially expressed genes:52</p>

<h3>Manr</h3>

<p>Same as with linc-Enc1a, may be extra-knockout region OR perhaps hets or some sort of artifact.. </p>

<p>Number of differentially expressed genes:49</p>

<h3>Peril</h3>

<p>Number of differentially expressed genes:215</p>

<h3>Trp53cor1 (linc-p21)</h3>

<p>Everything very lowly expressed </p>

<p>Number of differentially expressed genes:69</p>

<h3>Tug1 ** remove tug1_0 (JR828)</h3>

<p>Number of differentially expressed genes:67</p>

<h1>Embryonic Samples</h1>

<h3>Celrr</h3>

<p>Number of differentially expressed genes:39</p>

<h3>Crnde</h3>

<p>Number of differentially expressed genes:80 </p>

<h3>Eldr</h3>

<p>Number of differentially expressed genes:151</p>

<h3>Haunt</h3>

<p>Number of differentially expressed genes:33</p>

<h3>Kantr</h3>

<p>Number of differentially expressed genes:766</p>

<h3>Brn1a</h3>

<h3>Brn1b</h3>

<p>Number of differentially expressed genes:130</p>

<h3>linc-Cox2</h3>

<p>Number of differentially expressed genes:97</p>

<h3>linc-Enc1a</h3>

<p>Number of differentially expressed genes:16</p>

<h3>Manr</h3>

<p>Number of differentially expressed genes:86</p>

<h3>Peril</h3>

<p>Number of differentially expressed genes:254</p>

<h3>Trp53cor1 (linc-p21)</h3>

<pre><code>## Error: RS-DBI driver: (error in statement: near &quot;)&quot;: syntax error)
</code></pre>

<pre><code>## Error: error in evaluating the argument &#39;object&#39; in selecting a method for function &#39;annotation&#39;: Error: object &#39;DEgenes&#39; not found
</code></pre>

<p>Number of differentially expressed genes:0</p>

<h3>Tug1</h3>

<p>Number of differentially expressed genes:19</p>

<h1>SUMMARY</h1>

<p>Range of number of differentially expressed genes:</p>

<!-- html table generated in R 3.0.2 by xtable 1.7-3 package -->

<!-- Thu Jul  3 13:06:04 2014 -->

<TABLE border=1>
<TR> <TH>  </TH> <TH>       V1 </TH>  </TR>
  <TR> <TD align="right"> 1 </TD> <TD> 0      : 1   </TD> </TR>
  <TR> <TD align="right"> 2 </TD> <TD> 130    : 1   </TD> </TR>
  <TR> <TD align="right"> 3 </TD> <TD> 151    : 1   </TD> </TR>
  <TR> <TD align="right"> 4 </TD> <TD> 16     : 1   </TD> </TR>
  <TR> <TD align="right"> 5 </TD> <TD> 19     : 1   </TD> </TR>
  <TR> <TD align="right"> 6 </TD> <TD> 193    : 1   </TD> </TR>
  <TR> <TD align="right"> 7 </TD> <TD> (Other):45   </TD> </TR>
   </TABLE>

<p>Number of differentially expressed genes, by condition: </p>

<pre><code>## Error: formal argument &quot;breaks&quot; matched by multiple actual arguments
</code></pre>

<p><img src="figure/overview/diff%20summary.png" alt="plot of chunk diff summary"/> </p>

<p><img src="figure/overview/lacZ%20correlation.png" alt="plot of chunk lacZ correlation"/> </p>

<pre><code>## [1] 0.1647
</code></pre>

<pre><code>## character(0)
</code></pre>

</body>

</html>

