Mouse Data Overview DE genelists 
========================================================




# ADULT KNOCKOUT STRAINS 

### Celrr


Number of differentially expressed genes:54

### Crnde






Number of differentially expressed genes:42

### Eldr









Number of differentially expressed genes:27

### Haunt



Number of differentially expressed genes:37


### Kantr (Jarid1C)





Number of differentially expressed genes:315

### Brn1a
3 isoforms expressed



Number of differentially expressed genes:74


### Brn1b
LacZ expressed slightly higher than lincRNA



Number of differentially expressed genes:193


### linc-Cox2



Endogenous linc and LZ approx same expression levels 



Number of differentially expressed genes:94


### linc-Enc1a







Number of differentially expressed genes:52


### Manr
Same as with linc-Enc1a, may be extra-knockout region OR perhaps hets or some sort of artifact.. 





Number of differentially expressed genes:49


### Peril





Number of differentially expressed genes:215



### Trp53cor1 (linc-p21)
Everything very lowly expressed 



Number of differentially expressed genes:69


### Tug1 ** remove tug1_0 (JR828)



Number of differentially expressed genes:67


# Embryonic Samples

### Celrr







Number of differentially expressed genes:39


### Crnde






Number of differentially expressed genes:80 

### Eldr





Number of differentially expressed genes:151


### Haunt






Number of differentially expressed genes:33


### Kantr






Number of differentially expressed genes:766

### Brn1a


### Brn1b 



Number of differentially expressed genes:130

### linc-Cox2






Number of differentially expressed genes:97


### linc-Enc1a






Number of differentially expressed genes:16


### Manr 






Number of differentially expressed genes:86

### Peril






Number of differentially expressed genes:254


### Trp53cor1 (linc-p21)





```
## Error: RS-DBI driver: (error in statement: near ")": syntax error)
```

```
## Error: error in evaluating the argument 'object' in selecting a method for function 'annotation': Error: object 'DEgenes' not found
```

Number of differentially expressed genes:0

### Tug1





Number of differentially expressed genes:19

# SUMMARY

Range of number of differentially expressed genes:

<!-- html table generated in R 3.0.2 by xtable 1.7-3 package -->
<!-- Thu Jul  3 13:06:04 2014 -->
<TABLE border=1>
<TR> <TH>  </TH> <TH>       V1 </TH>  </TR>
  <TR> <TD align="right"> 1 </TD> <TD> 0      : 1   </TD> </TR>
  <TR> <TD align="right"> 2 </TD> <TD> 130    : 1   </TD> </TR>
  <TR> <TD align="right"> 3 </TD> <TD> 151    : 1   </TD> </TR>
  <TR> <TD align="right"> 4 </TD> <TD> 16     : 1   </TD> </TR>
  <TR> <TD align="right"> 5 </TD> <TD> 19     : 1   </TD> </TR>
  <TR> <TD align="right"> 6 </TD> <TD> 193    : 1   </TD> </TR>
  <TR> <TD align="right"> 7 </TD> <TD> (Other):45   </TD> </TR>
   </TABLE>


Number of differentially expressed genes, by condition: 


```
## Error: formal argument "breaks" matched by multiple actual arguments
```

![plot of chunk diff summary](figure/overview/diff summary.png) 



![plot of chunk lacZ correlation](figure/overview/lacZ correlation.png) 

```
## [1] 0.1647
```



```
## character(0)
```




