<!DOCTYPE html>
<!-- saved from url=(0014)about:internet -->
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>

<title>Trp53cor1 KO vs WT (Embryonic)</title>

<style type="text/css">
body, td {
   font-family: sans-serif;
   background-color: white;
   font-size: 12px;
   margin: 8px;
}

tt, code, pre {
   font-family: 'DejaVu Sans Mono', 'Droid Sans Mono', 'Lucida Console', Consolas, Monaco, monospace;
}

h1 { 
   font-size:2.2em; 
}

h2 { 
   font-size:1.8em; 
}

h3 { 
   font-size:1.4em; 
}

h4 { 
   font-size:1.0em; 
}

h5 { 
   font-size:0.9em; 
}

h6 { 
   font-size:0.8em; 
}

a:visited {
   color: rgb(50%, 0%, 50%);
}

pre {	
   margin-top: 0;
   max-width: 95%;
   border: 1px solid #ccc;
   white-space: pre-wrap;
}

pre code {
   display: block; padding: 0.5em;
}

code.r, code.cpp {
   background-color: #F8F8F8;
}

table, td, th {
  border: none;
}

blockquote {
   color:#666666;
   margin:0;
   padding-left: 1em;
   border-left: 0.5em #EEE solid;
}

hr {
   height: 0px;
   border-bottom: none;
   border-top-width: thin;
   border-top-style: dotted;
   border-top-color: #999999;
}

@media print {
   * { 
      background: transparent !important; 
      color: black !important; 
      filter:none !important; 
      -ms-filter: none !important; 
   }

   body { 
      font-size:12pt; 
      max-width:100%; 
   }
       
   a, a:visited { 
      text-decoration: underline; 
   }

   hr { 
      visibility: hidden;
      page-break-before: always;
   }

   pre, blockquote { 
      padding-right: 1em; 
      page-break-inside: avoid; 
   }

   tr, img { 
      page-break-inside: avoid; 
   }

   img { 
      max-width: 100% !important; 
   }

   @page :left { 
      margin: 15mm 20mm 15mm 10mm; 
   }
     
   @page :right { 
      margin: 15mm 10mm 15mm 20mm; 
   }

   p, h2, h3 { 
      orphans: 3; widows: 3; 
   }

   h2, h3 { 
      page-break-after: avoid; 
   }
}

</style>





</head>

<body>
<h1>Trp53cor1 KO vs WT (Embryonic)</h1>

<pre><code>## Error: cannot open the connection
</code></pre>

<pre><code>## Error: object &#39;deletionCoords&#39; not found
</code></pre>

<p>This file shows the wt-v-ko comparison for Trp53cor1. </p>

<p>Cuff overview:</p>

<pre><code>## CuffSet instance with:
##   0 samples
##   0 genes
##   0 isoforms
##   0 TSS
##   0 CDS
##   0 promoters
##   0 splicing
##   0 relCDS
</code></pre>

<h1>QC</h1>

<h2>Dispersion</h2>

<p>Dispersion plot for genes in cuff:
(Overdispersion can lead to innacurate quants)</p>

<pre><code>## Error: replacement has 1 row, data has 0
</code></pre>

<h2>Cross-replicate variability (fpkmSCVplot)</h2>

<p>Differences in CV 2 can result in lower numbers of differentially expressed genes due to a higher degree of variability between replicate fpkm estimates.</p>

<p>Genes:</p>

<pre><code>## Error: RS-DBI driver: (error in statement: near &quot;)&quot;: syntax error)
</code></pre>

<p>Isoforms: </p>

<pre><code>## Error: RS-DBI driver: (error in statement: near &quot;)&quot;: syntax error)
</code></pre>

<h2>MvA plot</h2>

<pre><code>## Error: non-numeric argument to binary operator
</code></pre>

<h3>MvA plot counts</h3>

<pre><code>## Error: non-numeric argument to binary operator
</code></pre>

<h2>Scatterplot matrix</h2>

<pre><code>## Error: RS-DBI driver: (error in statement: near &quot;from&quot;: syntax error)
</code></pre>

<h2>Distributions</h2>

<h3>Boxplots</h3>

<p>Boxplot (genes)</p>

<pre><code>## Error: RS-DBI driver: (error in statement: near &quot;)&quot;: syntax error)
</code></pre>

<p>Boxplot (genes, replicates)</p>

<pre><code>## Error: RS-DBI driver: (error in statement: near &quot;)&quot;: syntax error)
</code></pre>

<p>Boxplot (isoforms)</p>

<pre><code>## Error: RS-DBI driver: (error in statement: near &quot;)&quot;: syntax error)
</code></pre>

<p>Boxplot (isoforms, replicates)</p>

<pre><code>## Error: RS-DBI driver: (error in statement: near &quot;)&quot;: syntax error)
</code></pre>

<h3>Density</h3>

<p>Density (genes)</p>

<pre><code>## Error: RS-DBI driver: (error in statement: near &quot;)&quot;: syntax error)
</code></pre>

<p>Density (genes, replicates)</p>

<pre><code>## Error: RS-DBI driver: (error in statement: near &quot;)&quot;: syntax error)
</code></pre>

<h2>Clustering</h2>

<h3>Replicate Clusters</h3>

<pre><code>## Error: RS-DBI driver: (error in statement: near &quot;from&quot;: syntax error)
</code></pre>

<h3>PCA (genes)</h3>

<pre><code>## Error: RS-DBI driver: (error in statement: near &quot;from&quot;: syntax error)
</code></pre>

<h3>MDS (genes)</h3>

<pre><code>## Error: RS-DBI driver: (error in statement: near &quot;from&quot;: syntax error)
</code></pre>

<h1>KO assessment</h1>

<h2>Endogenous lncRNA expression</h2>

<pre><code>## Error: RS-DBI driver: (error in statement: near &quot;)&quot;: syntax error)
</code></pre>

<pre><code>## Error: error in evaluating the argument &#39;object&#39; in selecting a method for function &#39;expressionPlot&#39;: Error: object &#39;myGene&#39; not found
</code></pre>

<p>Endogenous expression of Trp53cor1 isoforms:</p>

<pre><code>## Error: error in evaluating the argument &#39;object&#39; in selecting a method for function &#39;expressionPlot&#39;: Error in isoforms(myGene) : 
##   error in evaluating the argument &#39;object&#39; in selecting a method for function &#39;isoforms&#39;: Error: object &#39;myGene&#39; not found
</code></pre>

<p>Barplot of gene expression:</p>

<pre><code>## Error: error in evaluating the argument &#39;object&#39; in selecting a method for function &#39;expressionBarplot&#39;: Error: object &#39;myGene&#39; not found
</code></pre>

<p>Barplot of isoform expression:</p>

<pre><code>## Error: error in evaluating the argument &#39;object&#39; in selecting a method for function &#39;expressionBarplot&#39;: Error in isoforms(myGene) : 
##   error in evaluating the argument &#39;object&#39; in selecting a method for function &#39;isoforms&#39;: Error: object &#39;myGene&#39; not found
</code></pre>

<h2>LacZ expression</h2>

<pre><code>## Error: RS-DBI driver: (RS_SQLite_exec: could not execute1: cannot start a
## transaction within a transaction)
</code></pre>

<pre><code>## Error: error in evaluating the argument &#39;object&#39; in selecting a method for function &#39;expressionPlot&#39;: Error: object &#39;myGene&#39; not found
</code></pre>

<pre><code>## Error: error in evaluating the argument &#39;object&#39; in selecting a method for function &#39;expressionBarplot&#39;: Error: object &#39;myGene&#39; not found
</code></pre>

<h2>Digital Genotyping (LacZ vs Endogenous lncRNA and Sex)</h2>

<p>Eif2s3y is a y-expressed gene, Xist is an x-expressed gene 
Expression plot (endogenous linc, lacZ, Y-expressed gene):</p>

<pre><code>## Error: RS-DBI driver: (RS_SQLite_exec: could not execute1: cannot start a
## transaction within a transaction)
</code></pre>

<pre><code>## Error: error in evaluating the argument &#39;object&#39; in selecting a method for function &#39;expressionBarplot&#39;: Error: object &#39;genotypingGenes&#39; not found
</code></pre>

<p>Expression heatmap:</p>

<pre><code>## Error: error in evaluating the argument &#39;object&#39; in selecting a method for function &#39;csHeatmap&#39;: Error: object &#39;genotypingGenes&#39; not found
</code></pre>

<h3>Track visualization</h3>

<pre><code>## Error: RS-DBI driver: (RS_SQLite_exec: could not execute1: cannot start a
## transaction within a transaction)
</code></pre>

<pre><code>## Error: error in evaluating the argument &#39;object&#39; in selecting a method for function &#39;annotation&#39;: Error: object &#39;myGene&#39; not found
</code></pre>

<pre><code>## Error: object &#39;annot&#39; not found
</code></pre>

<pre><code>## Error: error in evaluating the argument &#39;x&#39; in selecting a method for function &#39;unlist&#39;: Error: object &#39;locus&#39; not found
</code></pre>

<pre><code>## Error: object &#39;locus&#39; not found
</code></pre>

<pre><code>## Error: object &#39;locus&#39; not found
</code></pre>

<pre><code>## Error: error in evaluating the argument &#39;x&#39; in selecting a method for function &#39;unlist&#39;: Error: object &#39;start_and_end&#39; not found
</code></pre>

<pre><code>## Error: object &#39;start_and_end&#39; not found
</code></pre>

<pre><code>## Error: object &#39;start_and_end&#39; not found
</code></pre>

<pre><code>## Error: error in evaluating the argument &#39;start&#39; in selecting a method for function &#39;.buildRange&#39;: Error: object &#39;from&#39; not found
</code></pre>

<pre><code>## Error: cannot open the connection
</code></pre>

<pre><code>## Error: object &#39;deletionCoords&#39; not found
</code></pre>

<pre><code>## Error: object &#39;deletionCoords&#39; not found
</code></pre>

<pre><code>## Error: non-character argument
</code></pre>

<pre><code>## Error: object of type &#39;closure&#39; is not subsettable
</code></pre>

<pre><code>## Error: object of type &#39;closure&#39; is not subsettable
</code></pre>

<pre><code>## Error: error in evaluating the argument &#39;x&#39; in selecting a method for function &#39;unlist&#39;: Error: object &#39;positions&#39; not found
</code></pre>

<pre><code>## Error: object &#39;positions&#39; not found
</code></pre>

<pre><code>## Error: object &#39;positions&#39; not found
</code></pre>

<pre><code>## Error: unique() applies only to vectors
</code></pre>

<h1>Differential Analysis</h1>

<h2>Differential Genes</h2>

<pre><code>## Error: RS-DBI driver: (error in statement: near &quot;)&quot;: syntax error)
</code></pre>

<pre><code>## Error: error in evaluating the argument &#39;object&#39; in selecting a method for function &#39;annotation&#39;: Error: object &#39;sigGenes&#39; not found
</code></pre>

<p>[1] &ldquo;No sig genes!&rdquo;</p>

<h3>Matrix of gene significant differences between conditions</h3>

<p>(skip for Brainmap wt-v-ko comparisons)</p>

<p><img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAASAAAAEgCAMAAAAjXV6yAAACylBMVEUAAAABAQECAgIDAwMEBAQFBQUGBgYHBwcICAgJCQkKCgoLCwsMDAwNDQ0ODg4PDw8QEBARERESEhITExMUFBQVFRUWFhYXFxcYGBgZGRkaGhobGxscHBwdHR0eHh4fHx8gICAhISEiIiIjIyMkJCQlJSUmJiYnJycoKCgpKSkqKiorKyssLCwuLi4vLy8wMDAxMTEyMjIzMzM1NTU2NjY3Nzc4ODg5OTk6Ojo7Ozs9PT0+Pj4/Pz9BQUFCQkJDQ0NERERFRUVHR0dISEhJSUlKSkpLS0tMTExNTU1OTk5PT09RUVFSUlJTU1NUVFRWVlZYWFhaWlpbW1tcXFxdXV1eXl5fX19gYGBhYWFiYmJlZWVmZmZnZ2doaGhpaWlqampra2tsbGxtbW1ubm5vb29wcHBxcXFzc3N0dHR1dXV2dnZ3d3d4eHh5eXl6enp7e3t+fn5/f3+AgICBgYGCgoKEhISFhYWGhoaHh4eIiIiJiYmKioqLi4uOjo6Pj4+QkJCTk5OUlJSVlZWWlpaXl5eYmJiZmZmampqbm5ucnJydnZ2enp6goKChoaGjo6OkpKSlpaWmpqanp6epqamqqqqrq6usrKytra2urq6vr6+w/5qxsbGysrKzs7O0tLS1tbW2tra3t7e4uLi6urq7u7u8vLy9vb2+vr6+/6y/v7/AwMDA/67BwcHCwsLDw8PD/7LExMTFxcXGxsbIyMjJycnLy8vL/7zMzMzQ0NDQ/8LR0dHT09PT/8fU1NTV1dXW1tbX19fY2NjY/83Z2dna2trb29vc3Nzd3d3e3t7e/9Xf39/g4ODh4eHi4uLj4+Pj/9zk5OTl5eXl/97m5ubn5+fo6Ojp6enq6urr6+vr/+bs7Ozt7e3u7u7v7+/w8PDx8fHy8vLz8/P09PT19fX29vb39/f4+Pj5+fn6+vr7+/v8/Pz9/f3+/v7///9s0QY2AAAACXBIWXMAAAsSAAALEgHS3X78AAAJvElEQVR4nO3d/39VdQHH8c92d4EN3GQDhkG6aEIMhYFEfgkkVlLBSEvMMISUmYCYg1IXZYaUTsjJCs3Gl6kBQWhgkduIurSNrbHRSMNV7NsF3b279/M/dM6995xz79mX922dc++F+379cDmfe+4597On9+sHeUxINmwi3hNI9AgEIhCIQCACgQgEIhCIQCACgQgEIhCIQCACgQgEIhCIQCACgQgEIhCIQCACgQgEIhCIQCACgQgEIhCIQCCrgD56bGrq5LXd6hldA+/E5S92DronmLfDolnYkFVAa2dVN1TP/qKytfHCgJ0bLzSJxkH3BHMl8OPYqqld+7ZycVoM9VAYniAZgMbsVi78hy8Fnkgv5TtmH1X+FPtvS8t/U70qVTgCe7zrs8fc1SqPfTYj7WZ1R/AGgb1q2oHy1Rsdc/bq+/Xx67McN+yyaMZRZhXQKuc9v+gOntFVlfHCmaobVaC5x9+/f6p6VeAxolzznelv1d35md5xi2rqH86V2g20R5B+4DtZL9c/lnpQP0Fo3Dv2mabto3ssmnJ0WQXk3bk03fmlFvWMrlufVv6oUoFekrJOhAF5r1UeDq4l/3y2QcoadUfwBhqQfuDyrcrGXXfqJwiNm8Q56d/rtmjK0WXhs//DPXMKfCrD6BPKqEcFOhl8fdGBmkXwp+vft/ne69UdwRtoQPqBk95VNn48Xj9BaOxblPW1ig+tm3E0WQT0XrFXvRTNKkOa+uO4VSCXCahJBH48f9FNZa+9KaR2Aw1IPzBFqKXoJ9DG/W9862O39Fsz5SizCMiddkC5bEn5j/pDF6pPiH2DAXnGHpGyccrvhXK7EwOB9AMnHVc2Wuv0E4TGHaVSfiCarJlylFn1FFtz3c53qxcUq2d0vZyxo2lfgTg9EEiWzHz71JIFnc6fdR4rFB8YQO8HTqIfuP7mgy2v5W7STxAaXx77XPOz11y2aMrRZRWQ54nrU3JWdckAQ3meY/47onUQoI8eHj/mC21y53Vptx39VJ52A/fMUcGzaAf2bchNzdvi1U+gjXd/InV6tUUzjjI7PqJV1SgXJ0d5YnegjdkBtPGmmv76hV+J4YE2ZgeQ+6sO4Vg2gi+gIz7Qxuz5FnS59VKMD7StBP6amBjZCzRgoSdsSWjI1aHEyl6gAesYAMhTkplZ4o3cXq1+iP6yfXMEJRbQk3lvHb3hqcjtRasPHz580r45gmwCCi346As9+gKQ8pGxakr6feq6UcVcx7RqY5da3i7l4/S0yO2pe+yZYpTZA6Qt+GiPIGMBSAGae+zQtLXKZv4fOh6YYuxSuijapTwnusK33aJoXObXu22ZZjTZA3QxtOCjAV3UF4AUIOVr7f5xyteICuVTszB2KTUI5UO0R12+NrZrxOOuA7Pi9+HRpqdYaMFHfw3SF4AUIOXLZq9o0dd6tF1KZ4TyotwnGsK3Perj6URK3B5C9gBpCz4aUNgCUADILZq1L6L6LqUu8Z6U7aLTvK1AxXaNIyx7gFpCCz4aUIuxABR4ir2R4dGA9F1qn3xFyt3TIra3r/BLWTc6xuuIRvYAaQs+2kJP2AKQSxQcP5L/oL5Ypu9S21pw+s8zy6Qs7dC3Tzk3uY7MKbFlmtFk02tQaMFHX+gxFoBc4oWJ6Ssv6UD6LjXvuqyxD3mD+7TtA/PTJm6I3wpIzL+LJfJfEg4WgUAxn27z7Fjf4/8XADq2/4rtR9asFgCgX1tyJ3HplRpLTkMgEIFABAIRCEQgEIFABAIRCEQgEIFABAIRCEQgEIFABAIRCEQgEIFABAIRCEQgEIFABAIRCEQgEIFABAIRCEQgEIFABAIRCEQgEIFABAIRCEQgEIFABAIRCEQgEIFABAIRCEQgUNIAucRy5yMjOE0SAf38J84RnCaJgPx/Gck/r0wioJH9c1kCgZIGaKQRCEQgEIFASQP011pT/4ruNEkD9JtfmfpbdKchEIhAIAKBCAQiEIhAIAKBCAQiEIhAoAigU3NG3X1J9oZdR6AIoHk/dT+6hkARRQBl+mV/YT2BwosA+vhJKV8v6iJQWBFAO7IekvKRuQQKK/Jd7IwC4j+02biCQIO/zevXEohAIAKBCAQiEIhAIAKB+G0eZAI6d7vz7DfCfgsKgUxAy54W7mX3GWMCmYCyvUL2TjDGBDIBTfAL2ZdjjAlkAlq6T/Ssu8cYE8gE1L7YmfMAX6TD4ts8KBxIaBlXEYiPIBCBQCaghqKM9CWNxphAJqDCLZ1dm+cZYwKZgDJ9UvoyjTGBTECbvt/TU7bWGBPIBKS90VeGxlc70Nk7br29ddjTJPm72IpyWV487GmSHGh8h+zIHvY0JqAnnFfrJ+k/1db+ztCpra39u3ptik/2pwx7GhNQVp0/Ynz1AA3xCLoo/50z6OFaJqDppt1XO9CKClkx/G8iNwE9usMXMb7agVoXFi1qG/Y0JqAN/DZvygSU05pcr0E4E9C6Q8n1FMMN+knaGBMoyT8o4gYB8q00tglkAvqeQ3mGzTfGBDIBTWz5ZtOrW4wxgcx/9Sy37fPPMMYEMgHNePHAmvbJxphAJqDKAs/CMc8YYwLxbR5EIJAJaOvC3jnjthtjApnfxc7vWtmWa4wJZALKbVu+9wL/D7OwTEDbMhb05fFdLCy+SIMIBCIQiEAgAoEIBCIQiEAgAoEIBCIQiEAgAoEIBCIQiEAgAoEIBCIQiEAgAoEIBEoaoD/+1tQ/ojtN0gCNNAKBCAQiEIhAIAKBCAQiEIhAIAKBCAQiEIhAIAKBCAQiEIhAIAKBCAQiEIhAIAKBCAQiEIhAIAKBCAQiEIhAIAKBCAQiEIhAIAKBCAQiEIhAIAKBCAQiEIhAIAKBCAQiEAgBlV651ccCyOe5crPEBwExAoEIBCIQiEAgAoEIBCIQiECguAPFfQKguM9v4AR8M+MwjSFLPKDnF8d9TuHFYDLbJ+f8QMrKGeMnPancX0nO+m/nZ21Wth7PnNeoTqB96YQJxe36zQ/tTTagjLbzC6UsKJfNDuX+DraKcn+NU9kq85V+Xp3A4j1+z1N3xHZO0ReDyXy6uNItpffgc6uVOxM+v+iXfnWrR3ZmqBMI/GpXZ2znFH0xmEzfnnsLpfzcg3sbReD+RPBuFaCudHUrp1dK99nYzin6YjCZvAvnxkmZft7/S+ELByrzfXeJurWqpM+9/P7Yzin6YjCZH2ZP2qZc5s4ovWVlONCaawob1K3uu7MnrOyO7ZyiL36TSSiGoUscIGH+DZ2JUfwmVBm3e/6fSrj/YokWgUAEAhEIRCAQgUAEAhEIRCAQgUAEAhEIRCAQgUAEAhEIRCAQgUAEAv0X8Yu4oAP6qyEAAAAASUVORK5CYII=" alt="plot of chunk sigMatrix"/> </p>

<h3>Significant gene expression differences between conditions</h3>

<p>Expression plot (genes):(turned off)</p>

<pre><code>## Error: error in evaluating the argument &#39;object&#39; in selecting a method for function &#39;csHeatmap&#39;: Error: object &#39;sigGenes&#39; not found
</code></pre>

<p>Significant genes with expression &gt;50fpkm (any condition):(turned off)</p>

<p>An individual look at each of the highly expressed significantly differentially regulated genes:
(eval=false for first pass)</p>

<h3>Expression-level/significance relationship</h3>

<p>Scatter plot of significant genes only:</p>

<pre><code>## Error: error in evaluating the argument &#39;object&#39; in selecting a method for function &#39;csScatter&#39;: Error: object &#39;sigGenes&#39; not found
</code></pre>

<p>Volcano Plot</p>

<pre><code>## Error: One or more values of &#39;x&#39; or &#39;y&#39; are not valid sample names!
</code></pre>

<p>Volcano plot with significant genes only:</p>

<pre><code>## Error: error in evaluating the argument &#39;object&#39; in selecting a method for function &#39;csVolcano&#39;: Error: object &#39;sigGenes&#39; not found
</code></pre>

<p>FoldChange Heatmap</p>

<pre><code>## Error: error in evaluating the argument &#39;object&#39; in selecting a method for function &#39;csFoldChangeHeatmap&#39;: Error: object &#39;sigGenes&#39; not found
</code></pre>

<h2>Differential Splicing</h2>

<h3>Differential Isoforms between conditions</h3>

<p>Per isoform difference between conditions:</p>

<p><img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAASAAAAEgCAMAAAAjXV6yAAAC01BMVEUAAAABAQECAgIDAwMEBAQFBQUGBgYHBwcICAgJCQkKCgoLCwsMDAwNDQ0ODg4PDw8QEBARERESEhITExMUFBQVFRUWFhYXFxcYGBgZGRkaGhobGxscHBwdHR0eHh4fHx8gICAhISEiIiIjIyMkJCQlJSUmJiYnJycoKCgpKSkqKiorKyssLCwtLS0uLi4vLy8wMDAxMTEyMjIzMzM1NTU2NjY3Nzc4ODg5OTk6Ojo7Ozs9PT0+Pj4/Pz9BQUFCQkJDQ0NERERFRUVHR0dISEhJSUlKSkpLS0tMTExNTU1OTk5PT09RUVFSUlJTU1NUVFRWVlZYWFhaWlpbW1tcXFxdXV1eXl5fX19gYGBhYWFiYmJjY2NlZWVmZmZnZ2doaGhpaWlqampra2tsbGxtbW1ubm5vb29wcHBxcXFzc3N0dHR1dXV2dnZ3d3d4eHh5eXl6enp7e3t+fn5/f3+AgICBgYGCgoKEhISFhYWGhoaHh4eIiIiJiYmKioqLi4uOjo6Pj4+QkJCTk5OVlZWWlpaXl5eYmJiZmZmampqbm5ucnJydnZ2enp6goKChoaGjo6OkpKSlpaWmpqanp6epqamqqqqrq6usrKytra2urq6vr6+w/5qxsbGysrKzs7O0tLS1tbW2tra3t7e4uLi5ubm6urq7u7u8vLy9vb2+vr6+/6y/v7/AwMDA/67BwcHCwsLDw8PD/7LExMTFxcXGxsbHx8fIyMjJycnLy8vL/7zMzMzQ0NDQ/8LR0dHT09PT/8fU1NTV1dXW1tbX19fY2NjY/83Z2dna2trb29vc3Nzd3d3e3t7e/9Xf39/g4ODh4eHi4uLj4+Pj/9zk5OTl5eXl/97m5ubn5+fo6Ojp6enq6urr6+vr/+bs7Ozt7e3u7u7v7+/w8PDx8fHy8vLz8/P09PT19fX29vb39/f4+Pj5+fn6+vr7+/v8/Pz9/f3+/v7///8Jn5YvAAAACXBIWXMAAAsSAAALEgHS3X78AAAKGUlEQVR4nO3dj3/UdQHH8Q9sB+wHmzAQDNJJBG2jYCCRPwIJKioYaIlhGD9UZippAqUSZSKlCIlQagUMMMM1QsMScoOIIza2xkYjFVexsd3A3e3u8yf0/dzd977f++7H+9rdfe/g3q/Hw/H93ue773329L7344OIkKzPRKInkOwRCEQgEIFABAIRCEQgEIFABAIRCEQgEIFABAIRCEQgEIFABAIRCEQgEIFABAIRCEQgEIFABAIRCEQgEIFA0QJ99N2xA0evvKjO5Ox+cqevxNHjSCBPc+hAy4j/+5KiaIFWFu2t3jv5K9rW6vPdBlefrxU1PY4EcorQgZYR//clRdECXfOW9uWkaO5l2Nnn+Xsf7fv77CzaiQx5Wfviq+jwXyYvjk+bfFD7Vey5JX386+qmgSLNP+J5ePiQLzfIQ5/PTP+MGggc4B/1z8Ip9xWl3bBdSteSrOx7OgIjwW0pygpWSrFpbOa6fZ8eOOZVqR9rS9ECLXXc+cuLgTM5d2Y+f3rnJxXQ1LffXzJW3eR/JGi3fG/Cm0dv/1x79qzKUw+MkvoBoceJcLZnPV27aXCbXFb053eKVgRGgttSFG08IcXtDc+JSYc/WDJa6sfaUrRAnq3zMhxfrVdnct78lPbLTgX0opRHhQnIc432sHHO+eCZaikr1UDgABNQrTgrfbtdnRnlUpZnedSIvq09eNQhFdItfuv/tuCxUc48wmJwrV/eNaXQq37IwUe0vTYFdCzwLBICqhOBH6erbO1d16uBwAEmIO+s3G9suyxPi0vapSXq1Yi+LUWlOuRE4B9tJHisPUUJ9F6JR30VdeqHTH9Xqp/I6X/VDgOqFf6fxzd30vpfvy6kfoAJSHa9dt/HbuqqUygdok6N6NuBNwHqi37ewLHRzTzSogRype/XvtYP+K+affEGbbusJyB31gEpa8a8I7TjjvQI1LxGyg9FrTuzQl1WbjWib3cDCh4b3cwjLdpLbMV1W9/dO6NEncn5UuaW2rJCcbI7kCwteOv4nBktjp+3HCoWHxpA7wdn4byU9WzdM0MvyeWTDh8uui9AF9zuBqQfa0vRArkfv35A3tJW6Z/95vy06X8SDT0AffTAsCFfapRbr0u/5eCn8vUDXAWDgrNwypdvHDhhr3ZJfWvo0KWXA0DB7e6XWPBYW4rlG7Kd6sn02CB3DE+Z+GIJtHpSZdepmYtieMYkKJZArq+nibT5vX3quEKL7WeeSw0dMT1fEpQ0HwqTtfgCeazXm2nhp9dVouQqvkDdVi0AkLs0J6fUE769TGh9LX5zBCUX0BP5bx684cnw7VnLKioqjsVvjqA4AQUXfkILPqGFIO2t484xGXerVZ5tU9PG7TWGVPnbpXxpXPj22F3xmWKExQdIX/jRH0HGQpAGNPVQ+biV2ub4w833jjGGtC6IJinPilbztkvMzc755sW4TDOS4gN0IbjwowNdCC0EaUDax9s92R7tEaS97RbGkFa10N6FuwPL0fp2pXjMub9oUVymGUlxusSCCz+h56DQQpAGpH3KbFerPMFFI31IqhUg7Um5U1Sbt93q8XRkQMIeQvEB0hd+dCDTQpAzsAxWp3/yDA1ptYr3pGwSLdZtDcqmxY3uxQeoPrjwowPVGwtB/kvstUy3DhQaUn3iFe2j+riw7U0LfVIeHWzXAmK34gOkL/zoCz6mhSCnKHz7wPjlobWL0JBqQ+HJvxWsl3JNc2j7uONR54EppXGZZiTF6TkouPATWvAxFoKc4vmRGYs7jMUdfUjlWZWbdb8nMKZv75+ePvKRxC2h2P5ZLHl+SzCyCASyfbp1k+2+x+gCQIf2XLH9JDarBQDodzG5k4T0SmVMTkMgEIFABAIRCEQgEIFABAIRCEQgEIFABAIRCEQgEIFABAIRCEQgEIFABAIRCEQgEIFABAIRCEQgEIFABAIRCEQgEIFABAIRCEQgEIFABAIRCEQgEIFABAIRCEQgEIFABAIRCEQgUMoAOcUCx4P9OE0KAf3ip/35PwynEJDvRH/+eGUKAfXvj8sSCJQyQP2NQCACgQgEShmgv1dZ+ndkp0kZoN//xtI/IjsNgUAEAhEIRCAQgUAEAhEIRCAQgUAEAoUBHZ8y6I4O2W66jUBhQNN+5npoBYHCCgPK8cmu4lMEMhcG9PFjUu6b20ogU2FAW3Lvl/LBqQQyFf4qdloD8ZWvNW4gUM8v86FbCUQgEIFABAIRCEQgEIFA/DQPsgCdvdVx5h7T34JCIAvQ/KeEa/7dxj6BLEDDPUK2jzD2CWQBGuETsjPP2CeQBWhemWhbdaexTyALUNNsR969fJI2xZd5kBlI6Bk3EYiPIBCBQBag6rmZGXNqjH0CWYCK17W0rp1m7BPIApTjldKbY+wTyAL06A/b2tavNPYJZAHSX+h3BPevdqAzt918a0Ofp0nxV7GFm+Xmkj5Pk+JAw5pl8/A+T2MBetxxtb6T/mtV1R8Nnaqqqn+qWwd4ZdeAPk9jAco96gvbv3qAenkEXZD/yevx2/UsQBMsw1c70MJtctuiPk9jAXpoizds/2oHapg5d1Zjn6exAD3CT/OWLEB5Dan1HISzAK0qT61LDNfjO2ljn0Ap/kYR1wOQd7GxTSAL0A/StCtsurFPIAvQyPpv1766ztgnkPW3nuXGMt9EY59AFqCJL+xf0TTa2CeQBWhHoXvmkKeNfQLxZR5EIJAFaMPM9inZm4x9Allfxc5tX9w4ytgnkAVoVOOC3ef5X5iZsgBtzJzRmc9XMVN8kgYRCEQgEIFABAIRCEQgEIFABAIRCEQgEIFABAIRCEQgEIFABAIRCEQgEIFABAIRCJQyQH/5g6V/RXaalAHqbwQCEQhEIBCBQAQCEQhEIBCBQAQCEQhEIBCBQAQCEQhEIBCBQAQCEQhEIBCBQAQCEQhEIBCBQAQCEQhEIBCBQAQCEQhEIBCBQAQCEQhEIBCBQAQCEQhEIBCBQAQCEQhEIBACWnPldsoOIK/7yi0mPgiIEQhEIBCBQAQCEQhEIBCBQAQCJRwo4RMAJXx+3SfgLUjANHot+YCem53wOZmzYTKbRuf9SModE4dd+4R2f6V5D39nfO5abeuxnGk1agJN80aMKGkKHV6+O9WAMhvPzZSycLOsS9Pu740GsdlX6dC21nvXfFFNYPYun/vJ2+ydU+TZMJnPluxwSel549ll2p0Jr090SZ/aapMtmWoC/r/a1WHvnCLPhsl07rqrWMovLN9dI/z3JwJ3qwG1ZqitvHYpXWfsnVPk2TCZ/PNns6XMOOf7lfCagdZ7vz9HbS0t7XQtWGLvnCLPhsn8ePi1G7WvoyauuWmxGWjF0OJqtXXxjuEjFl+0d06Rl7jJJBVD7yUPkLD+DZ3JUeImtCNh9/x/lXT/xpItAoEIBCIQiEAgAoEIBCIQiEAgAoEIBCIQiEAgAoEIBCIQiEAgAoEIBPofA6BTNbi0/6oAAAAASUVORK5CYII=" alt="plot of chunk diff_iso"/> </p>

<p>These isoforms are:</p>

<p>[1] &ldquo;no sig isoforms&rdquo;</p>

<p>Gene-level DE isoform heatmap</p>

<pre><code>## [1] &quot;no sig isoforms&quot;
</code></pre>

<p>Isoform foldchange heatmap by isoform:</p>

<pre><code>## [1] &quot;no sig isoforms&quot;
</code></pre>

<h3>Differential Splicing between conditions</h3>

<p>(eval false for first pass)</p>

<p>Per condition differences in isoforms (Does gene have diff piechart between conditions?)</p>

<p>These genes are:</p>

<p>Splicing heatmap by isoform:</p>

<p>Splicing heatmap by gene</p>

<p>The following are significantly differentially spliced genes (relative portion of isoform per condition): </p>

<h1>Gene/Pathway Analysis</h1>

<h2>GSEA</h2>

<p>Enrichment and zscores are calculated based on expression in KO vs WT (fpkmKO/fpkmWT), so genes that are down regulated in KO are shown in blue, while upregulation is shown in red. </p>

<p>KO/WT
Blue = down in KO
Red = Up in KO</p>

<pre><code>## Error: Only alternative=&quot;mixed&quot; is possible with F-like statistics.
</code></pre>

<pre><code>## Error: object &#39;reactome_pvl_mat&#39; not found
</code></pre>

<pre><code>## Error: object &#39;reactome_pvl_corrected&#39; not found
</code></pre>

<pre><code>## Error: Only alternative=&quot;mixed&quot; is possible with F-like statistics.
</code></pre>

<pre><code>## Error: object &#39;biocarta_pvl_mat&#39; not found
</code></pre>

<pre><code>## Error: object &#39;biocarta_pvl_corrected&#39; not found
</code></pre>

<pre><code>## Error: Only alternative=&quot;mixed&quot; is possible with F-like statistics.
</code></pre>

<pre><code>## Error: object &#39;kegg_pvl_mat&#39; not found
</code></pre>

<pre><code>## Error: object &#39;kegg_pvl_corrected&#39; not found
</code></pre>

<pre><code>## Error: Only alternative=&quot;mixed&quot; is possible with F-like statistics.
</code></pre>

<pre><code>## Error: object &#39;interneuron_pvl_mat&#39; not found
</code></pre>

<pre><code>## Error: object &#39;interneuron_pvl_corrected&#39; not found
</code></pre>

<pre><code>## Error: Only alternative=&quot;mixed&quot; is possible with F-like statistics.
</code></pre>

<pre><code>## Error: object &#39;oncogene_pvl_mat&#39; not found
</code></pre>

<pre><code>## Error: object &#39;oncogene_pvl_corrected&#39; not found
</code></pre>

<pre><code>## Error: Only alternative=&quot;mixed&quot; is possible with F-like statistics.
</code></pre>

<pre><code>## Error: object &#39;immuno_pvl_mat&#39; not found
</code></pre>

<pre><code>## Error: object &#39;immuno_pvl_corrected&#39; not found
</code></pre>

<pre><code>## Error: Only alternative=&quot;mixed&quot; is possible with F-like statistics.
</code></pre>

<pre><code>## Error: object &#39;tf_pvl_mat&#39; not found
</code></pre>

<pre><code>## Error: object &#39;tf_pvl_corrected&#39; not found
</code></pre>

<p>Biocarta enrichment: </p>

<pre><code>## Error: error in evaluating the argument &#39;x&#39; in selecting a method for function &#39;t&#39;: Error: object &#39;biocarta_pvl_corrected&#39; not found
</code></pre>

<pre><code>## Error: object &#39;x&#39; not found
</code></pre>

<pre><code>## Error: object &#39;x_ordered&#39; not found
</code></pre>

<pre><code>## Error: object &#39;x_ordered&#39; not found
</code></pre>

<pre><code>## Error: object &#39;noinfinities_x&#39; not found
</code></pre>

<pre><code>## Error: object &#39;x_max&#39; not found
</code></pre>

<pre><code>## Error: error in evaluating the argument &#39;x&#39; in selecting a method for function &#39;as.matrix&#39;: Error: object &#39;x_ordered&#39; not found
</code></pre>

<pre><code>## Error: object &#39;x_ordered&#39; not found
</code></pre>

<p>Biocarta zscore: </p>

<pre><code>## Error: error in evaluating the argument &#39;x&#39; in selecting a method for function &#39;which&#39;: Error in apply(x, 2, min) : object &#39;biocarta_pvl_corrected&#39; not found
## Calls: colMins -&gt; apply
</code></pre>

<pre><code>## Error: object &#39;x&#39; not found
</code></pre>

<pre><code>## Error: object &#39;x_ordered&#39; not found
</code></pre>

<pre><code>## Error: error in evaluating the argument &#39;x&#39; in selecting a method for function &#39;as.matrix&#39;: Error: object &#39;x_ordered&#39; not found
</code></pre>

<pre><code>## Error: object &#39;x_ordered&#39; not found
</code></pre>

<p>Reactome enrichment: </p>

<pre><code>## Error: error in evaluating the argument &#39;x&#39; in selecting a method for function &#39;t&#39;: Error: object &#39;reactome_pvl_corrected&#39; not found
</code></pre>

<pre><code>## Error: object &#39;x&#39; not found
</code></pre>

<pre><code>## Error: object &#39;x_ordered&#39; not found
</code></pre>

<pre><code>## Error: object &#39;x_ordered&#39; not found
</code></pre>

<pre><code>## Error: object &#39;noinfinities_x&#39; not found
</code></pre>

<pre><code>## Error: object &#39;x_max&#39; not found
</code></pre>

<pre><code>## Error: object &#39;x_ordered&#39; not found
</code></pre>

<p>Reactome zscore: </p>

<pre><code>## Error: error in evaluating the argument &#39;x&#39; in selecting a method for function &#39;which&#39;: Error in apply(x, 2, min) : object &#39;reactome_pvl_corrected&#39; not found
## Calls: colMins -&gt; apply
</code></pre>

<pre><code>## Error: object &#39;x&#39; not found
</code></pre>

<pre><code>## Error: object &#39;x_ordered&#39; not found
</code></pre>

<pre><code>## Error: error in evaluating the argument &#39;x&#39; in selecting a method for function &#39;as.matrix&#39;: Error: object &#39;x_ordered&#39; not found
</code></pre>

<pre><code>## Error: object &#39;x_ordered&#39; not found
</code></pre>

<p>Kegg enrichment: </p>

<pre><code>## Error: error in evaluating the argument &#39;x&#39; in selecting a method for function &#39;t&#39;: Error: object &#39;kegg_pvl_corrected&#39; not found
</code></pre>

<pre><code>## Error: object &#39;x&#39; not found
</code></pre>

<pre><code>## Error: object &#39;x_ordered&#39; not found
</code></pre>

<pre><code>## Error: object &#39;x_ordered&#39; not found
</code></pre>

<pre><code>## Error: object &#39;noinfinities_x&#39; not found
</code></pre>

<pre><code>## Error: object &#39;x_max&#39; not found
</code></pre>

<pre><code>## Error: object &#39;x_ordered&#39; not found
</code></pre>

<p>Kegg zscore: </p>

<pre><code>## Error: error in evaluating the argument &#39;x&#39; in selecting a method for function &#39;which&#39;: Error in apply(x, 2, min) : object &#39;kegg_pvl_corrected&#39; not found
## Calls: colMins -&gt; apply
</code></pre>

<pre><code>## Error: object &#39;x&#39; not found
</code></pre>

<pre><code>## Error: object &#39;x_ordered&#39; not found
</code></pre>

<pre><code>## Error: error in evaluating the argument &#39;x&#39; in selecting a method for function &#39;as.matrix&#39;: Error: object &#39;x_ordered&#39; not found
</code></pre>

<pre><code>## Error: object &#39;x_ordered&#39; not found
</code></pre>

<p>Interneuron enrichment:</p>

<pre><code>## Error: error in evaluating the argument &#39;x&#39; in selecting a method for function &#39;t&#39;: Error: object &#39;interneuron_pvl_corrected&#39; not found
</code></pre>

<pre><code>## Error: object &#39;x&#39; not found
</code></pre>

<pre><code>## Error: object &#39;x_ordered&#39; not found
</code></pre>

<pre><code>## Error: object &#39;x_ordered&#39; not found
</code></pre>

<pre><code>## Error: object &#39;noinfinities_x&#39; not found
</code></pre>

<pre><code>## Error: object &#39;x_max&#39; not found
</code></pre>

<pre><code>## Error: object &#39;x_ordered&#39; not found
</code></pre>

<p>Interneuron zscore:</p>

<pre><code>## Error: error in evaluating the argument &#39;x&#39; in selecting a method for function &#39;which&#39;: Error in apply(x, 2, min) : object &#39;interneuron_pvl_corrected&#39; not found
## Calls: colMins -&gt; apply
</code></pre>

<pre><code>## Error: object &#39;x&#39; not found
</code></pre>

<pre><code>## Error: object &#39;x_ordered&#39; not found
</code></pre>

<pre><code>## Error: error in evaluating the argument &#39;x&#39; in selecting a method for function &#39;as.matrix&#39;: Error: object &#39;x_ordered&#39; not found
</code></pre>

<pre><code>## Error: object &#39;x_ordered&#39; not found
</code></pre>

<p>Oncogene enrichment: </p>

<pre><code>## Error: error in evaluating the argument &#39;x&#39; in selecting a method for function &#39;t&#39;: Error: object &#39;oncogene_pvl_corrected&#39; not found
</code></pre>

<pre><code>## Error: object &#39;x&#39; not found
</code></pre>

<pre><code>## Error: object &#39;x_ordered&#39; not found
</code></pre>

<pre><code>## Error: object &#39;x_ordered&#39; not found
</code></pre>

<pre><code>## Error: object &#39;noinfinities_x&#39; not found
</code></pre>

<pre><code>## Error: object &#39;x_max&#39; not found
</code></pre>

<pre><code>## Error: object &#39;x_ordered&#39; not found
</code></pre>

<p>Oncogene zscore:</p>

<pre><code>## Error: error in evaluating the argument &#39;x&#39; in selecting a method for function &#39;which&#39;: Error in apply(x, 2, min) : object &#39;oncogene_pvl_corrected&#39; not found
## Calls: colMins -&gt; apply
</code></pre>

<pre><code>## Error: object &#39;x&#39; not found
</code></pre>

<pre><code>## Error: object &#39;x_ordered&#39; not found
</code></pre>

<pre><code>## Error: error in evaluating the argument &#39;x&#39; in selecting a method for function &#39;as.matrix&#39;: Error: object &#39;x_ordered&#39; not found
</code></pre>

<pre><code>## Error: object &#39;x_ordered&#39; not found
</code></pre>

<p>Immuno enrichment:</p>

<pre><code>## Error: error in evaluating the argument &#39;x&#39; in selecting a method for function &#39;t&#39;: Error: object &#39;immuno_pvl_corrected&#39; not found
</code></pre>

<pre><code>## Error: object &#39;x&#39; not found
</code></pre>

<pre><code>## Error: object &#39;x_ordered&#39; not found
</code></pre>

<pre><code>## Error: object &#39;x_ordered&#39; not found
</code></pre>

<pre><code>## Error: object &#39;noinfinities_x&#39; not found
</code></pre>

<pre><code>## Error: object &#39;x_max&#39; not found
</code></pre>

<pre><code>## Error: object &#39;x_ordered&#39; not found
</code></pre>

<p>Immuno zscore:</p>

<pre><code>## Error: error in evaluating the argument &#39;x&#39; in selecting a method for function &#39;which&#39;: Error in apply(x, 2, min) : object &#39;immuno_pvl_corrected&#39; not found
## Calls: colMins -&gt; apply
</code></pre>

<pre><code>## Error: object &#39;x&#39; not found
</code></pre>

<pre><code>## Error: object &#39;x_ordered&#39; not found
</code></pre>

<pre><code>## Error: error in evaluating the argument &#39;x&#39; in selecting a method for function &#39;as.matrix&#39;: Error: object &#39;x_ordered&#39; not found
</code></pre>

<pre><code>## Error: object &#39;x_ordered&#39; not found
</code></pre>

<p>TF enrichment:</p>

<pre><code>## Error: error in evaluating the argument &#39;x&#39; in selecting a method for function &#39;t&#39;: Error: object &#39;tf_pvl_corrected&#39; not found
</code></pre>

<pre><code>## Error: object &#39;x&#39; not found
</code></pre>

<pre><code>## Error: object &#39;x_ordered&#39; not found
</code></pre>

<pre><code>## Error: object &#39;x_ordered&#39; not found
</code></pre>

<pre><code>## Error: object &#39;noinfinities_x&#39; not found
</code></pre>

<pre><code>## Error: object &#39;x_max&#39; not found
</code></pre>

<pre><code>## Error: object &#39;x_ordered&#39; not found
</code></pre>

<p>TF zscore:</p>

<pre><code>## Error: error in evaluating the argument &#39;x&#39; in selecting a method for function &#39;which&#39;: Error in apply(x, 2, min) : object &#39;tf_pvl_corrected&#39; not found
## Calls: colMins -&gt; apply
</code></pre>

<pre><code>## Error: object &#39;x&#39; not found
</code></pre>

<pre><code>## Error: object &#39;x_ordered&#39; not found
</code></pre>

<pre><code>## Error: error in evaluating the argument &#39;x&#39; in selecting a method for function &#39;as.matrix&#39;: Error: object &#39;x_ordered&#39; not found
</code></pre>

<pre><code>## Error: object &#39;x_ordered&#39; not found
</code></pre>

<h2>GO enrichment</h2>

<p>Cluster profiler used to call enichments of significantly differentially regulated genes that map to Entrez IDs. </p>

<pre><code>## Error: RS-DBI driver: (error in statement: near &quot;)&quot;: syntax error)
</code></pre>

<pre><code>## Error: error in evaluating the argument &#39;object&#39; in selecting a method for function &#39;annotation&#39;: Error: object &#39;sigGenes&#39; not found
</code></pre>

<pre><code>## Error: object &#39;geneAnnot&#39; not found
</code></pre>

<pre><code>## Error: error in evaluating the argument &#39;object&#39; in selecting a method for function &#39;diffData&#39;: Error: object &#39;sigGenes&#39; not found
</code></pre>

<pre><code>## Error: object &#39;sigDiff&#39; not found
</code></pre>

<pre><code>## Error: object &#39;geneNames&#39; not found
</code></pre>

<pre><code>## Error: object &#39;sigEntrez&#39; not found
</code></pre>

<pre><code>## Error: error in evaluating the argument &#39;x&#39; in selecting a method for function &#39;unlist&#39;: Error: object &#39;sigEZ&#39; not found
</code></pre>

<pre><code>## Error: object &#39;sigEZ&#39; not found
</code></pre>

<pre><code>## Error: object &#39;sigEZ&#39; not found
</code></pre>

<pre><code>## Error: object &#39;sigEZ&#39; not found
</code></pre>

<pre><code>## Error: object &#39;sigEZ&#39; not found
</code></pre>

<pre><code>## Error: object &#39;sigEZ&#39; not found
</code></pre>

<pre><code>## Error: error in evaluating the argument &#39;x&#39; in selecting a method for function &#39;plot&#39;: Error: object &#39;goBP&#39; not found
</code></pre>

<pre><code>## Error: error in evaluating the argument &#39;x&#39; in selecting a method for function &#39;plot&#39;: Error: object &#39;goMF&#39; not found
</code></pre>

<pre><code>## Error: error in evaluating the argument &#39;x&#39; in selecting a method for function &#39;plot&#39;: Error: object &#39;goCC&#39; not found
</code></pre>

<pre><code>## Error: error in evaluating the argument &#39;x&#39; in selecting a method for function &#39;plot&#39;: Error: object &#39;kegg&#39; not found
</code></pre>

<pre><code>## Error: error in evaluating the argument &#39;x&#39; in selecting a method for function &#39;plot&#39;: Error: object &#39;pathway&#39; not found
</code></pre>

<h3>Enrichment or depletion for stage-specific cell cycle markers</h3>

<pre><code>## Error: RS-DBI driver: (RS_SQLite_exec: could not execute1: cannot start a
## transaction within a transaction)
</code></pre>

<pre><code>## Error: error in evaluating the argument &#39;object&#39; in selecting a method for function &#39;annotation&#39;: Error: object &#39;cellcycle_genes&#39; not found
</code></pre>

<pre><code>## Error: error in evaluating the argument &#39;object&#39; in selecting a method for function &#39;diffData&#39;: Error: object &#39;cellcycle_genes&#39; not found
</code></pre>

<pre><code>## Error: error in evaluating the argument &#39;x&#39; in selecting a method for function &#39;merge&#39;: Error: object &#39;diffDat&#39; not found
</code></pre>

<pre><code>## Error: object &#39;cellcycle&#39; not found
</code></pre>

<pre><code>## Error: error in evaluating the argument &#39;x&#39; in selecting a method for function &#39;merge&#39;: Error: object &#39;cellcycle&#39; not found
</code></pre>

<pre><code>## Error: object &#39;cellcycle&#39; not found
</code></pre>

<pre><code>## Error: object &#39;cellcycle&#39; not found
</code></pre>

<pre><code>## Error: object &#39;cellcycle.melt&#39; not found
</code></pre>

<pre><code>## Error: object &#39;cellcycle.melt&#39; not found
</code></pre>

<pre><code>## Error: object &#39;cellcycle.melt&#39; not found
</code></pre>

<pre><code>## Error: object &#39;cellcycle.p&#39; not found
</code></pre>

<pre><code>## Error: object &#39;cellcycle.p&#39; not found
</code></pre>

<h3>Enrichment or depletion for markers of specific cortical layers</h3>

<pre><code>## Error: RS-DBI driver: (RS_SQLite_exec: could not execute1: cannot start a
## transaction within a transaction)
</code></pre>

<pre><code>## Error: error in evaluating the argument &#39;object&#39; in selecting a method for function &#39;annotation&#39;: Error: object &#39;cortical_genes&#39; not found
</code></pre>

<pre><code>## Error: error in evaluating the argument &#39;object&#39; in selecting a method for function &#39;diffData&#39;: Error: object &#39;cortical_genes&#39; not found
</code></pre>

<pre><code>## Error: error in evaluating the argument &#39;x&#39; in selecting a method for function &#39;merge&#39;: Error: object &#39;diffDat&#39; not found
</code></pre>

<pre><code>## Error: object &#39;cortical&#39; not found
</code></pre>

<pre><code>## Error: error in evaluating the argument &#39;x&#39; in selecting a method for function &#39;merge&#39;: Error: object &#39;cortical&#39; not found
</code></pre>

<pre><code>## Error: object &#39;cortical&#39; not found
</code></pre>

<pre><code>## Error: object &#39;cortical&#39; not found
</code></pre>

<pre><code>## Error: object &#39;cortical.melt&#39; not found
</code></pre>

<pre><code>## Error: object &#39;cortical.melt&#39; not found
</code></pre>

<pre><code>## Error: object &#39;cortical.melt&#39; not found
</code></pre>

<pre><code>## Error: object &#39;cortical.p&#39; not found
</code></pre>

<pre><code>## Error: object &#39;cortical.p&#39; not found
</code></pre>

<pre><code>## Error: object &#39;cortical.p&#39; not found
</code></pre>

<h3>Enrichment or depletion for specific neural cell types</h3>

<pre><code>## Error: RS-DBI driver: (RS_SQLite_exec: could not execute1: cannot start a
## transaction within a transaction)
</code></pre>

<pre><code>## Error: error in evaluating the argument &#39;object&#39; in selecting a method for function &#39;annotation&#39;: Error: object &#39;neuralcell_genes&#39; not found
</code></pre>

<pre><code>## Error: error in evaluating the argument &#39;object&#39; in selecting a method for function &#39;diffData&#39;: Error: object &#39;neuralcell_genes&#39; not found
</code></pre>

<pre><code>## Error: error in evaluating the argument &#39;x&#39; in selecting a method for function &#39;merge&#39;: Error: object &#39;diffDat&#39; not found
</code></pre>

<pre><code>## Error: object &#39;neuralcelltype&#39; not found
</code></pre>

<pre><code>## Error: error in evaluating the argument &#39;x&#39; in selecting a method for function &#39;merge&#39;: Error: object &#39;neuralcelltype&#39; not found
</code></pre>

<pre><code>## Error: object &#39;neuralcelltype&#39; not found
</code></pre>

<pre><code>## Error: object &#39;neuralcelltype&#39; not found
</code></pre>

<pre><code>## Error: object &#39;neuralcelltype&#39; not found
</code></pre>

<pre><code>## Error: object &#39;neuralcelltype.melt&#39; not found
</code></pre>

<pre><code>## Error: object &#39;neuralcelltype.melt&#39; not found
</code></pre>

<pre><code>## Error: object &#39;neuralcelltype.p&#39; not found
</code></pre>

<pre><code>## Error: object &#39;neuralcelltype.p&#39; not found
</code></pre>

<pre><code>## Error: object &#39;neuralcelltype.p&#39; not found
</code></pre>

<h3>Enrichment or Depletion of neural differentiation markers</h3>

<pre><code>## Error: RS-DBI driver: (RS_SQLite_exec: could not execute1: cannot start a
## transaction within a transaction)
</code></pre>

<pre><code>## Error: error in evaluating the argument &#39;object&#39; in selecting a method for function &#39;annotation&#39;: Error: object &#39;neural_diff_genes&#39; not found
</code></pre>

<pre><code>## Error: error in evaluating the argument &#39;object&#39; in selecting a method for function &#39;diffData&#39;: Error: object &#39;neural_diff_genes&#39; not found
</code></pre>

<pre><code>## Error: error in evaluating the argument &#39;x&#39; in selecting a method for function &#39;merge&#39;: Error: object &#39;diffDat&#39; not found
</code></pre>

<pre><code>## Error: object &#39;ndiff&#39; not found
</code></pre>

<pre><code>## Error: error in evaluating the argument &#39;x&#39; in selecting a method for function &#39;merge&#39;: Error: object &#39;ndiff&#39; not found
</code></pre>

<pre><code>## Error: object &#39;ndiff&#39; not found
</code></pre>

<pre><code>## Error: object &#39;ndiff&#39; not found
</code></pre>

<pre><code>## Error: object &#39;ndiff.melt&#39; not found
</code></pre>

<pre><code>## Error: object &#39;ndiff.melt&#39; not found
</code></pre>

<pre><code>## Error: object &#39;ndiff.melt&#39; not found
</code></pre>

<pre><code>## Error: object &#39;ndiff.p&#39; not found
</code></pre>

<pre><code>## Error: object &#39;ndiff.p&#39; not found
</code></pre>

<pre><code>## Error: object &#39;ndiff.p&#39; not found
</code></pre>

<h1>Cis vs Trans (locally)</h1>

<p>log2 Foldchange and test statistic are calculated with the ratio of fpkm(KO)/fpkm(WT), thus the test_stat is positive if a gene is higher in the KO and negative if a gene has lower expression in the KO</p>

<pre><code>## Error: dims [product 1] do not match the length of object [0]
</code></pre>

<pre><code>## Error: object &#39;fullTable&#39; not found
</code></pre>

<pre><code>## Error: object &#39;myGene&#39; not found
</code></pre>

<pre><code>## Error: object &#39;myGene&#39; not found
</code></pre>

<pre><code>## Error: object &#39;myGene&#39; not found
</code></pre>

<pre><code>## Error: object &#39;fullTable&#39; not found
</code></pre>

<pre><code>## Error: error in evaluating the argument &#39;x&#39; in selecting a method for function &#39;nrow&#39;: Error in empty(.data) : object &#39;sigGenesRegion&#39; not found
## Calls: ddply -&gt; empty
</code></pre>

<pre><code>## Error: object &#39;fullTable&#39; not found
</code></pre>

<pre><code>## Error: object &#39;fullTable&#39; not found
</code></pre>

<pre><code>## Error: object &#39;myGene&#39; not found
</code></pre>

<pre><code>## Error: object &#39;genesInRegion&#39; not found
</code></pre>

<pre><code>## Error: object &#39;genesInRegion&#39; not found
</code></pre>

<pre><code>## Error: object &#39;genesInRegion&#39; not found
</code></pre>

<pre><code>## Error: object &#39;genesInRegion&#39; not found
</code></pre>

<pre><code>## Error: object of type &#39;closure&#39; is not subsettable
</code></pre>

<pre><code>## Error: object of type &#39;closure&#39; is not subsettable
</code></pre>

<pre><code>## Error: object of type &#39;closure&#39; is not subsettable
</code></pre>

<pre><code>## Error: object of type &#39;closure&#39; is not subsettable
</code></pre>

<pre><code>## Error: object of type &#39;closure&#39; is not subsettable
</code></pre>

<pre><code>## Error: ggplot2 doesn&#39;t know how to deal with data of class function
</code></pre>

<p>The pvalue for &mdash;&ndash; genes significantly regulated in a region this size  is: &mdash;&ndash;</p>

<pre><code>## Error: object &#39;currplot&#39; not found
</code></pre>

<h1>Notes</h1>

<h2>Samples used are:</h2>

<!-- html table generated in R 3.0.2 by xtable 1.7-3 package -->

<!-- Thu Sep 25 16:16:59 2014 -->

<TABLE border=1>
<TR> <TH>  </TH> <TH> NA </TH>  </TR>
  </TABLE>

<h2>Replicates</h2>

<!-- html table generated in R 3.0.2 by xtable 1.7-3 package -->

<!-- Thu Sep 25 16:16:59 2014 -->

<TABLE border=1>
<TR> <TH>  </TH> <TH> file </TH> <TH> sample_name </TH> <TH> replicate </TH> <TH> rep_name </TH> <TH> total_mass </TH> <TH> norm_mass </TH> <TH> internal_scale </TH> <TH> external_scale </TH>  </TR>
  </TABLE>

<h2>Session Info</h2>

<pre><code>## R version 3.0.2 (2013-09-25)
## Platform: x86_64-unknown-linux-gnu (64-bit)
## 
## locale:
##  [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
##  [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
##  [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
##  [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
##  [9] LC_ADDRESS=C               LC_TELEPHONE=C            
## [11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       
## 
## attached base packages:
## [1] grid      parallel  stats     graphics  grDevices utils     datasets 
## [8] methods   base     
## 
## other attached packages:
##  [1] plyr_1.8.1                         
##  [2] stringr_0.6.2                      
##  [3] seqbias_1.10.0                     
##  [4] BSgenome.Mmusculus.UCSC.mm10_1.3.19
##  [5] BSgenome_1.30.0                    
##  [6] Biostrings_2.30.1                  
##  [7] gdata_2.13.3                       
##  [8] org.Mm.eg.db_2.10.1                
##  [9] clusterProfiler_1.13.1             
## [10] DOSE_2.0.0                         
## [11] ReactomePA_1.6.1                   
## [12] AnnotationDbi_1.24.0               
## [13] Biobase_2.22.0                     
## [14] RMySQL_0.9-3                       
## [15] RColorBrewer_1.0-5                 
## [16] gridExtra_0.9.1                    
## [17] gtable_0.1.2                       
## [18] marray_1.40.0                      
## [19] gplots_2.14.2                      
## [20] GSA_1.03                           
## [21] limma_3.18.13                      
## [22] xtable_1.7-3                       
## [23] cummeRbund_2.7.2                   
## [24] Gviz_1.6.0                         
## [25] rtracklayer_1.22.7                 
## [26] GenomicRanges_1.14.4               
## [27] XVector_0.2.0                      
## [28] IRanges_1.20.7                     
## [29] fastcluster_1.1.13                 
## [30] reshape2_1.4                       
## [31] ggplot2_1.0.0                      
## [32] RSQLite_0.11.4                     
## [33] DBI_0.2-7                          
## [34] BiocGenerics_0.8.0                 
## [35] knitr_1.6                          
## 
## loaded via a namespace (and not attached):
##  [1] biomaRt_2.18.0         biovizBase_1.10.8      bitops_1.0-6          
##  [4] caTools_1.17.1         cluster_1.15.2         colorspace_1.2-4      
##  [7] dichromat_2.0-0        digest_0.6.4           DO.db_2.7             
## [10] evaluate_0.5.5         formatR_1.0            Formula_1.1-2         
## [13] GenomicFeatures_1.14.5 GO.db_2.10.1           GOSemSim_1.20.3       
## [16] graph_1.40.1           graphite_1.8.1         gtools_3.4.1          
## [19] Hmisc_3.14-4           igraph_0.7.1           KEGG.db_2.10.1        
## [22] KernSmooth_2.23-12     lattice_0.20-29        latticeExtra_0.6-26   
## [25] MASS_7.3-33            munsell_0.4.2          org.Hs.eg.db_2.10.1   
## [28] proto_0.3-10           qvalue_1.36.0          Rcpp_0.11.2           
## [31] RCurl_1.95-4.3         reactome.db_1.46.1     Rsamtools_1.14.3      
## [34] scales_0.2.4           splines_3.0.2          stats4_3.0.2          
## [37] survival_2.37-7        tcltk_3.0.2            tools_3.0.2           
## [40] XML_3.98-1.1           zlibbioc_1.8.0
</code></pre>

<h2>Run Info</h2>

<pre><code>##    param value
## 1 genome  mm10
</code></pre>

</body>

</html>

