`r strain` KO vs WT (Embryonic)
======================================

# questions
- overdispersion
- how do we rescale these plots? or just axes?
- add labels, descriptions
- turn off all caching 
- general figure formatting/table formatting 
- other heatmap labels (csHeatmap)


```{r init, echo=FALSE}
# Set knitr opts

path<-paste("figure/",strain,"/", sep="")
opts_chunk$set(fig.path=path, echo=FALSE)

### OPTIONAL VAR OVERRIDE (where are my variables!!?)
dir<-dir
strain<-strain
alpha<-alpha 

#setwd('/n/rinn_data1/users/agroff/GITHUB/BrainMap/analysis')
#setwd("/n/rinn_data1/users/agroff/seq/PERIL/data/diffs/cuffdiff_v221_newgtf/whole_brain/")

### THINGS TO CACHE GO HERE AND NOWHERE ELSE 

```

```{r vars_and_setup,echo=FALSE,results='hide',message=FALSE}
library(cummeRbund)
library(xtable)
library(limma)
library(GSA)
library(gplots)
library(marray)
library(ggplot2)
library(gtable)
library(gridExtra)


reactome_gs <- GSA.read.gmt("/n/rinn_data1/users/agroff/GSEA/c2.cp.reactome.v4.0.symbols.gmt")
biocarta_gs <- GSA.read.gmt("/n/rinn_data1/users/agroff/GSEA/c2.cp.biocarta.v4.0.symbols.gmt")

# Constant variables
alpha<-0.05

```

# Intialize
```{r cummeRbund and data setup, echo=FALSE}
setwd(dir)
cuff<-readCufflinks()
reps<-replicates(cuff)
files<-lapply(reps$file,function(x){strsplit(x, "/")})

#field 9 will be JR number 
files<-as.data.frame(files)
samples<-as.data.frame(t(files[9,]))

#look up strain, litter, sex, notes in master sheet 

LINCRNA<-strain

```

# Design Overview

This file shows the wt-v-ko comparison for `r strain`. 

Samples used are:
```{r results='asis'}
print(xtable(samples),type="html")
```


Cuff overview:
```{r}
cuff
```

## Replicates
```{r replicates, results='asis'}
print(xtable(replicates(cuff)),type="html")
```

# QC

## Dispersion

This shows count (WHAT IS COUNT? fragment counts? per gene?) vs dispersion (of those counts), or the spread in measurements for a particular gene across replicates. 

from cummerbund manual: Overdispersion is a common problem in RNA-Seq data. As of cufflinks v2.0 mean counts, variance, and dispersion are all emitted, allowing you to visualize the estimated overdispersion for each sample as a quality control measure. - See more at: http://compbio.mit.edu/cummeRbund/manual_2_0.html#sthash.9YRYxwRV.dpuf


```{r dispersion}
dispersionPlot(genes(cuff))
```

## Cross-replicate variability (fpkmSCVplot)
The squared coefficient of variation is a normalized measure of cross replicate variability that can be useful for evaluating the quality your RNA-seq data. Differences in CV 2 can result in lower numbers of differentially expressed genes due to a higher degree of variability between replicate fpkm estimates.

Genes:
```{r CV genes}
fpkmSCVPlot(genes(cuff))
```


Isoforms: 
```{r cv iso}
fpkmSCVPlot(isoforms(cuff))
```

## Volcano
```{r volcano}
csVolcano(genes(cuff),"wt","ko")
```

### Volcano matrix (replicates)
```{r volcano matrix}
csVolcanoMatrix(genes(cuff),replicates=T)
```

## MvA plot
```{r MvA}
MAplot(genes(cuff),"wt","ko")
```
   
### MvA plot counts
```{r MvA counts}
MAplot(genes(cuff),"wt","ko",useCount=T)
```

## Scatterplot
```{r scatterplot}
csScatterMatrix(genes(cuff))
```

### Scatter matrix (replicates)
```{r scatterplot matrix}
csScatterMatrix(genes(cuff),replicates=T)
```


## Distributions

### Boxplot
```{r boxplot}
csBoxplot(genes(cuff))
csBoxplot(genes(cuff),replicates=T)
csBoxplot(isoforms(cuff))
```

### Density
```{r density}
csDensity(genes(cuff))
csDensity(genes(cuff),replicates=T)
```


## Clustering

### Replicate Clusters
```{r replicate clusters}
csDendro(genes(cuff),replicates=T)
```

### PCA
```{r PCA}
PCAplot(genes(cuff),"PC2","PC3", replicates=T)
```

### MDS 
```{r MDS}
MDSplot(genes(cuff),replicates=T)
```

### Distance Heat Map
```{r R distance heatmap }
csDistHeat(genes(cuff), replicates=T)
```


# KO assessment

## Endogenous lncRNA expression
```{r Enodenous lncRNA tables, results='asis'}
myGeneID<-LINCRNA
myGene<-getGene(cuff, myGeneID)
print(xtable(fpkm(myGene)),type="html")
print(xtable(fpkm(isoforms(myGene))), type="html")
       
expressionPlot(myGene)
expressionPlot(myGene, replicates=TRUE)
       
expressionPlot(isoforms(myGene))
expressionPlot(isoforms(myGene), replicates=T)
       
expressionBarplot(myGene)
expressionBarplot(myGene,replicates=T)

expressionBarplot(isoforms(myGene))     
expressionBarplot(isoforms(myGene), replicates=T)

```


## LacZ expression
```{r LacZ expression, results='asis'}
myGeneID<-LacZ
myGene<-getGene(cuff, myGeneId)

print(xtable(fpkm(myGene)),type="html")
       
expressionPlot(myGene)
expressionPlot(myGene, replicates=TRUE)
             
expressionBarplot(myGene)
expressionBarplot(myGene,replicates=T)
```


## Digital Genotyping (LacZ vs Endogenous lncRNA and Sex)
```{r Digital Genotyping}
genotypingGeneIDs<-c(LINCRNA,"LacZ","Eif2s3y")
genotypingGenes<-getGenes(cuff,genotypingGeneIDs)
       
expressionBarplot(genotypingGenes)
expressionBarplot(genotypingGenes,replicates=T)
```


# Differential Analysis

## Differential Genes 

```{r differential genes}
sig<-getSig(cuff,alpha=alpha)       
sigGenes<-getGenes(cuff,sig)
geneAnnot<-annotation(sigGenes)
```

There are `r length(sig)` significantly differentially expressed genes. They are:

```{r sigdiffGenetable, results="asis"}
print(xtable(geneAnnot$gene_short_name),type="html")
```

### Matrix of gene significant differences between conditions
(skip for Brainmap wt-v-ko comparisons)

```{r sigMatrix}
sigMatrix(cuff, level="genes", alpha=alpha)
```

### Replicate Clustering by significant genes 
```{r sigDendro}
csDendro(sigGenes, replicates=T)
```

### Significant gene expression differences between conditions

```{r sigExpression}
csHeatmap(sigGenes, cluster="both",replicates=T)
expressionPlot(sigGenes)
expressionBarplot(sigGenes)
```

An individual look at each of the significantly differentially regulated genes:

```{r plot ALL OF THE SIGNIFICANT GENES, fig.height=(20 + (1*(length(sig)))), fig.width=(20 + (.5*length(sig)))}
plots<-lapply(sig,function(x){
  myGene<-getGene(cuff,x)
  return(expressionPlot(myGene,replicates=T))
  })
do.call(grid.arrange, plots)
```


### Expression-level significance relationship

**Interesting comments n stuff!**

```{r expression-sig relationship}
csScatter(sigGenes, "wt", "ko", smooth=T)
csVolcano(sigGenes, "wt", "ko")
```

## Differential Splicing

### Differential Isoforms between conditions
Per isoform difference between conditions:
```{r diff.iso,eval=TRUE,cache=TRUE}
sigMatrix(cuff, level="isoforms",alpha=alpha)

isoformSigIDs<-getSig(cuff,level="isoforms",alpha=alpha)
isoformSigGenes<-getGenes(cuff,isoformSigIDs)
isoAnnot<-annotation(isoformSigGenes)
```

These isoforms are:
```{r diff iso annotations, results='asis'}
print(xtable(isoAnnot$gene_short_name),type="html")
```

```{r isoform heatmap}
csHeatmap(isoforms(isoformSigGenes),cluster='row',method=dist)
```

### Differential Splicing between conditions

Per condition differences in isoforms (Does gene have diff piechart between conditions?)

```{r diff splicing}
splicingSigIDs<-getSig(cuff,level="splicing",alpha=alpha)
splicingSigGenes<-getGenes(cuff,splicingSigIDs)

sigMatrix(cuff, level='splicing', alpha=alpha)

spliceAnnot<-annotation(splicingSigGenes)
```

These genes are:
```{r diff splicing table of genes, results='asis'}
print(xtable(as.data.frame(spliceAnnot$gene_short_name)), type="html")
```

Splicing heatmap by isoform:
```{r splicing heatmap by isoform}
#pdf("sigSplicing_heatmap.pdf",width=10,height=20)
csHeatmap(isoforms(splicingSigGenes),cluster='row',method=dist)
#dev.off()
```

Splicing heatmap by gene
```{r splicing heatmap by gene}
csHeatmap(splicingSigGenes,cluster='row',method=dist)
```

The following are significantly differentially spliced genes (relative portion of isoform per condition): 

# how do we rescale these plots? or just axes?
```{r print all the splicing csPie plots, fig.height=(20 +1*length(splicingSigIDs)), fig.width=20}
csPiePlots<-lapply(splicingSigIDs,function(x){
  myGene<-getGene(cuff,x)
  csPie(myGene)
})
do.call(grid.arrange,csPiePlots)
```


```{r diff.TSS,eval=FALSE,echo=FALSE, results='hide'}
## Differential Promoter usage (isoforms by tss) <- NOT USING IN BRAINMAP

tssSigIDs<-getSig(cuff,alpha=alpha,level="TSS")
tssSigGenes<-getGenes(cuff,tssSigIDs)

csHeatmap(tssSigGenes,cluster='row',method=dist)

csPiePlots<-lapply(tssSigIDs,function(x){
  myGene<-getGene(cuff,x)
  csPie(myGene)
})
do.call(grid.arrange,csPiePlots)

```
 

```{r diff.promoters,eval=FALSE,echo=FALSE, results='hide'}
## Differential Promoter Usage (By "Promoters") <- NOT USING IN BRAINMAP

promoterSigIDs<-getSig(cuff,alpha=alpha,level="promoters")
promoterSigGenes<-getGenes(cuff,promoterSigIDs)

pdf("sigPromoter_heatmap.pdf",width=10,height=20)
csHeatmap(promoterSigGenes,cluster='row',method=dist)
dev.off()

csPiePlots<-lapply(promoterSigIDs,function(x){
  myGene<-getGene(cuff,x)
  csPie(myGene)
})
do.call(grid.arrange,csPiePlots)


```


```{r diff.CDS,eval=FALSE, echo=FALSE, results='hide'}
## Differential CDS? (unique protein coding isoforms) <-NOT IN BRAINMAP
cdsSigIDs<-getSig(cuff,alpha=alpha,level="CDS")
cdsSigGenes<-getGenes(cuff,cdsSigIDs)

relcdsSigIDs<-getSig(cuff,alpha=alpha,level="relCDS")
relcdsSigGenes<-getGenes(cuff,relcdsSigIDs)


pdf("sigCDS_heatmap.pdf",width=10,height=20)
csHeatmap(cdsSigGenes,cluster='row',method=dist)
dev.off()

pdf("sig_relCDS_heatmap.pdf",width=10,height=20)
csHeatmap(relcdsSigGenes,cluster='row',method=dist)
dev.off()
```


```{r overview.venn,echo=FALSE, eval=FALSE}
## Venn diagram overview of Differential changes


# Make venn diagram of DE genes, DE splicing, DE promoters, DE relCDS (See figure 2)

#install.packages("tiff")
#library(tiff)
#require(VennDiagram)
#venn<-venn.diagram(list(
#    'Genes'=sigGeneIDs,
#    'Splicing'=splicingSigIDs,
#    'Isoforms'=isoformSigIDs,
#    'Promoters'=promoterSigIDs,
#    'TSS'=tssSigIDs),filename=NULL)

#'CDS'=cdsSigIDs,
#'relCDS'=relcdsSigIDs

  
```


# Gene/Pathway Analysis

## GSEA

```{r gsea helper functions}
population<-genes(cuff)
population.diff<-diffData(population)
annotation<-annotation(genes(cuff))
gene_names<-merge(annotation,population.diff)

gene_set_index <- function(genelist, short_names){
  which(short_names %in% genelist)   
}

get_gene_set_p_vals <- function(input, gs, alternative){
  gene_set_indices <- lapply(gs$genesets, function(short_name){
    gene_set_index(input$short_name, short_name)
    })
  pvl<-lapply(gene_set_indices,geneSetTest,input$test_stat, alternative=alternative)
  pvl_mat<-as.data.frame(t(unlist(pvl)))
  colnames(pvl_mat) <- gs$geneset.names
   return(pvl_mat)
}
 
get_gene_set_q_vals <- function(pvl_mat, method="bonferroni"){
	comp_corrected <- matrix(p.adjust(pvl_mat, method=method), nrow=nrow(pvl_mat), ncol=ncol(pvl_mat))
	colnames(comp_corrected) <- colnames(pvl_mat)
	rownames(comp_corrected) <- rownames(pvl_mat)
	return(comp_corrected)
}

colMins<-function(x){
  apply(x,2,min)
}
rowMins<-function(x){
  apply(x,1,min)
}
 
InputCols<-maPalette(low="white",high="red",k=100)
```

```{r gsea}
df.pop<-data.frame("short_name"=toupper(gene_names$gene_short_name),"test_stat"=gene_names$test_stat)
row.names(df.pop)=population.diff$gene_id
df.pop.ordered<-df.pop[order(df.pop$test_stat),]
Input.df<-df.pop.ordered

reactome_pvl_mat <- get_gene_set_p_vals(Input.df, reactome_gs,alternative="mixed")
reactome_pvl_corrected <- get_gene_set_q_vals(reactome_pvl_mat)
reactome_pvl_corrected<-rbind(reactome_pvl_corrected,reactome_pvl_corrected)


biocarta_pvl_mat <- get_gene_set_p_vals(Input.df, biocarta_gs, alternative="mixed")
biocarta_pvl_corrected <- get_gene_set_q_vals(biocarta_pvl_mat)
biocarta_pvl_corrected<-rbind(biocarta_pvl_corrected,biocarta_pvl_corrected)
```

```{r print GSEA biocarta, fig.height=40, fig.width=10}
#two options, print as pdf, and adjust size in separate code block
#make minimum height and then increment scaled by number of rows to return 


setwd('/n/rinn_data1/users/agroff/GITHUB/BrainMap/analysis')

#print to file 
pdf("components_GSEA_biocarta.pdf",width=10,height=30)

#take top 50
x<-(-log10(t(biocarta_pvl_corrected[,which(colMins(biocarta_pvl_corrected) < 0.001)])))

x_ordered<-x[order(x[,1], decreasing=TRUE),]
x_ordered_subset<-x_ordered[1:50,]
noinfinities_x<-x_ordered_subset[which(x_ordered_subset!="Inf")]
x_max<-max(noinfinitiesx)+100
x_ordered_subset[x_ordered_subset=="Inf"]<-x_max


heatmap.2(x_ordered_subset, trace="none", margins=c(5,30), col=InputCols,dendrogram="row",lhei =c(0.1,0.9), labCol=c(""))

dev.off()

#print to report 
heatmap.2(x_ordered_subset, trace="none", margins=c(5,30), col=InputCols,dendrogram="row",lhei =c(0.1,0.9), labCol=c(""))
```

```{r print GSEA reactome, fig.width=10, fig.height=30}

#print to file
pdf("components_GSEA_reactome.pdf",width=10,height=30)

#take top 50 because this is ridiculous
#x<-(-log10(t(reactome_pvl_corrected[,which(colMins(reactome_pvl_corrected) < 0.0000000000000000000000001)])))

x<-(-log10(t(reactome_pvl_corrected[,which(colMins(reactome_pvl_corrected) < 0.0001)])))
x_ordered<-x[order(x[,1], decreasing=TRUE),]
x_ordered_subset<-x_ordered[1:50,]
noinfinities_x<-x_ordered_subset[which(x_ordered_subset!="Inf")]
x_max<-max(noinfinitiesx)+100
x_ordered_subset[x_ordered_subset=="Inf"]<-x_max


heatmap.2(x_ordered_subset, trace="none", margins=c(5,30),col=InputCols,dendrogram="both",lhei = c(0.1,0.90), labCol=c(""))
#cexRow = (0.2 + .5/log10(dim(x)[1]))
dev.off()

#print to report 
heatmap.2(x_ordered_subset, trace="none", margins=c(5,30),col=InputCols,dendrogram="both",lhei = c(0.1,0.90), labCol=c(""))

```

## GO enrichment 
Cluster profiler used to call enichments of significantly differentially regulated genes that map to Entrez IDs. 

Description/explanation of what's here, and justify all choices. 

```{r GO setup}
#source("http://bioconductor.org/biocLite.R")
#biocLite("ReactomePA")

library(ReactomePA)
library(DOSE)


sigGeneIDs<-getSig(cuff, alpha=0.05)
sigGenes<-getGenes(cuff,sigGeneIDs)
geneAnnot<-annotation(sigGenes)
geneNames<-geneAnnot$gene_short_name
sigDiff<-diffData(sigGenes)
sigDiff$foldChange<-sigDiff$value_1/sigDiff$value_2


# Top genes by test stat
#diff<-diffData(genes(cuff))
#diff<-diff[order(abs(diff$test_stat),decreasing=TRUE),]
#top_genes<-diff[which(diff$p_value<0.1),]
#top_genes_annot<-merge(top_genes,population)
#geneNames<-top_genes_annot$gene_short_name

```

biomart to get entrez gene IDS
clusterProfiler does GO enrichment 

BP, MF, CC
enrichKEGG
enrichPathway

```{r GO analysis setup and functions,fig.height=8,fig.width=8,echo=FALSE,eval=TRUE}
library(clusterProfiler)
require(biomaRt)
#source("http://bioconductor.org/biocLite.R")
#biocLite("clusterProfiler")
#biocLite("org.Mm.eg.db")
#http://www.bioconductor.org/packages/release/bioc/vignettes/biomaRt/inst/doc/biomaRt.pdf
ensembl <- useMart("ENSEMBL_MART_ENSEMBL","mmusculus_gene_ensembl", 
host="www.ensembl.org")

#Get entrezIDs
getEntrezIDs<-function (geneNames)
{
    tmp <- getBM(attributes = c("entrezgene"), filters = "mgi_symbol", values = geneNames, mart = ensembl)
    tmp
}
sigEntrez<-getEntrezIDs(geneNames)
sigEZ<-strsplit(as.character(sigEntrez), ", ")
sigEZ<-unlist(sigEZ)
require(org.Mm.eg.db) #mapping of entrez IDs to genbank
```

```{r GO enrichments, echo=FALSE}

goBP<-enrichGO(gene=sigEZ, organism="mouse",ont="BP",pvalueCutoff=0.01,readable=T)

goMF<-enrichGO(gene=sigEZ, organism="mouse",ont="MF",pvalueCutoff=0.01,readable=T)

goCC<-enrichGO(gene=sigEZ, organism="mouse",ont="CC",pvalueCutoff=0.01,readable=T)

kegg<-enrichKEGG(gene=sigEZ, organism="mouse",pvalueCutoff=1, readable=T)

pathway<-enrichPathway(gene=sigEZ,organism="mouse",pvalueCutoff=0.01, readable=T)

```

```{r GO figures, fig.height=8,fig.width=8, echo=FALSE}
#Visualizing w clusterProfiler
#barplot(goBP, drop = TRUE, showCategory = 12)
#barplot(goBP, showCategory = 12)
#barplot(goMF,showCategory=12)
#barplot(goCC, showCategory=12)

#WHY DONT THESE FUNCTIONS WORK? 
#cnetplot(goBP, categorySize = "pvalue", foldChange = geneList) #NEEDS foldChange values which requres exact entrez gene mapping (cant lose values); this requires a map. too much hassle for now. 
#cnetplot(goBP, categorySize = "geneNum", foldChange = geneList)

#cluster.enrichKEGG<-compareCluster(sigEZ,fun="enrichKEGG",organism="mouse",qvalueCutoff=0.01)
#require(ReactomePA)
#cluster.enrichPathway<-compareCluster(sigEntrezNames,fun="enrichPathway",qvalueCutoff=0.01)

plot(goBP,showCategory=10) + theme(axis.text.x=element_text(angle=-90,hjust=0)) + ggtitle("Enriched BP")
plot(goMF,showCategory=10) + theme(axis.text.x=element_text(angle=-90,hjust=0)) + ggtitle("Enriched MF")
plot(goCC,showCategory=10) + theme(axis.text.x=element_text(angle=-90,hjust=0)) + ggtitle("Enriched CC")

plot(kegg, showCategory=10) + ggtitle("Kegg Pathways")

plot(pathway,showCategory=10) + ggtitle("Reactome pathway enrichment")

#Loyal plots
#plot(cluster.enrichKEGG,showCategory=10) + theme(axis.text.x=element_text(angle=-90,hjust=0)) + ggtitle("Enriched KEGG")
#plot(cluster.enrichPathway,showCategory=20) + theme(axis.text.x=element_text(angle=-90,hjust=0)) + ggtitle("Enriched Reactome")
detach("package:biomaRt")
```

# Cis vs Trans (locally)
``` {r cis_v_trans}
window<-1000000

```



# Interesting Genes
Hand pick these after initial round of analysis?

# Notes

# Session Info
```{r session}
sessionInfo()
```

#Run Info
```{r runInfo}
runInfo(cuff)
```
