```{r standalone setup, echo=FALSE}
strain<-"linc-Enc1"
timepoint<-"Embryonic"
filename<-"linc-Enc1_vs_WT_Embryonic"
dir<-"/n/rinn_data1/seq/lgoff/Projects/BrainMap/data/diffs/linc-Enc1_vs_WT_Embryonic/"
```

`r strain` KO vs WT (`r timepoint`)
======================================

```{r init, echo=FALSE}
# Set knitr opts

#path<-paste(filename,"/figure/",sep="")
opts_chunk$set(echo=FALSE, message=FALSE, warning=FALSE,fig.height=4, fig.width=4)
alpha<-0.05 

analysisdir<-"/n/rinn_data1/users/agroff/GITHUB/BrainMap/analysis/"
diffdir<-"/n/rinn_data1/seq/lgoff/Projects/BrainMap/data/diffs"
GTF<-"/n/rinn_data1/seq/lgoff/Projects/BrainMap/data/annotation/mm10_gencode_vM2_with_lncRNAs_and_LacZ.gtf"
genome<-"mm10"

```

```{r vars_and_setup,echo=FALSE,results='hide',message=FALSE}
library(cummeRbund)
library(xtable)
library(limma)
library(GSA)
library(gplots)
library(marray)
library(ggplot2)


reactome_gs <- GSA.read.gmt("/n/rinn_data1/users/agroff/GSEA/c2.cp.reactome.v4.0.symbols.gmt")
biocarta_gs <- GSA.read.gmt("/n/rinn_data1/users/agroff/GSEA/c2.cp.biocarta.v4.0.symbols.gmt")

setwd(analysisdir)
deletionCoords<-read.table("mm10DeletionCoords.txt",sep="\t",header=TRUE,stringsAsFactors=FALSE)
colnames(deletionCoords)<-c("gene_name","gene_region","deletionRegion")


```

# Intialize
```{r cummeRbund_and_data_setup, echo=FALSE}
setwd(dir)
cuff<-readCufflinks(gtfFile=GTF,genome=genome) #rebuild=T until all first rounds done! build sep though bc issues :-/
reps<-replicates(cuff)
files<-lapply(reps$file,function(x){strsplit(x, "/")})

#field 9 will be JR number 
files<-as.data.frame(files)
samples<-(t(files[10,]))
rownames(samples)<-NULL

#look up strain, litter, sex, notes in master sheet 
```

# Design Overview

This file shows the wt-v-ko comparison for `r strain`. 

Cuff overview:
```{r}
cuff
```


# QC

## Dispersion

Dispersion plot for genes in cuff:
(Overdispersion can lead to innacurate quants)

```{r dispersion}
dispersionPlot(cuff)+theme_bw()
```

## Cross-replicate variability (fpkmSCVplot)
Differences in CV 2 can result in lower numbers of differentially expressed genes due to a higher degree of variability between replicate fpkm estimates.

Genes:
```{r CV_genes}
fpkmSCVPlot(genes(cuff))+theme_bw()
```

Isoforms: 
```{r cv_iso}
fpkmSCVPlot(isoforms(cuff))+theme_bw()
```
